```{r, include = FALSE}
ottrpal::set_knitr_image_path()
```

# Spatial transcriptomics

::: warning
This chapter chapter has currently been written by ChatGPT and has not been verified by experts. We need help writing and reviewing it! If you wish to contribute, please [go to this form](https://forms.gle/dqYgmKH8XXE2ohwD9) or our [GitHub page](https://github.com/fhdsl/Choosing_Genomics_Tools).
:::

## Learning Objectives

```{r, fig.alt = "This chapter will demonstrate how to: Approach collection of spatial transcriptomics data and design a typical analysis pipeline. Adjust your analysis pipeline to the research question, opportunities, and limitations concerning you spatial transcriptomics project. Learn about the questions that can be addressed with spatial transcriptomics data", out.width = "100%", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g15bed4cad37_396_1")
```

## What are the goals of spatial transcriptomic analysis?

Spatial transcriptomics (ST) technologies have been developed as a solution to the lack of spatial context in single cell transcriptomics (scRNA-seq) data [@rao2021exploring; @ospina2023primer]. There is a diversity of ST methods, however all have in common two features: Multiple measurements of gene expression and the locations within the tissue where those gene expression measurements were taken. Data analysis of ST data requires integration of those two components, and it's primary goal is to characterize gene expression patterns within the tissue or cellular context. The ability to quantify gene expression at different locations within the tissue is of tremendous value to understand the functional variation of different tissue regions, domains, or niches. It also places cell-cell communication in the context of cell neighborhoods, which ultimately facilitates a deeper understanding of cell and tissue biology, but also enables practical applications such as discovery of novel drug targets for complex diseases such as cancer [@dries2021advances; @williams2022introduction]. Following, are some of the specific goals that a study using ST could achieve:

1.  **Describe tissue-specific cellular neighborhoods of cell types and cell type sub-populations:** Although scRNA-seq continues to be a powerful method to assign biological identities to a mixture of cells, integrated analysis of ST combined with scRNA-seq adds crucial information to cell phenotypes by describing the neighborhoods where cells occur [@longo2021integrating]. Many methods to phenotype ST data are available, with most of them relying on the availability of a curated (scRNA-seq) cell type reference. Once cell identities have been determined, clustering or spatial statistics can be applied to describe the composition of tissue niches or domains. The explosion of ST data has resulted on novel and comprehensive tissue- or disease-specific atlases, not only describing the cell types within organs, but also the functional cell-cell relationships that result from spatial organization (e.g., @guilliams2022spatial; @wu2021single).
2.  **Uncover spatially regulated biological processes:** With ST data, there comes the ability to detect genes or gene pathways that are expressed in specific areas within tissues (i.e., spatially-restricted expression). Detecting genes with spatially-restricted expression is key to achieve further understanding of specific biological processes, such as tissue gradients, cell differentiation, or signaling pathways. For example, cancer researchers are now able to study signaling pathways restricted to the tumor-stroma interface [@hunter2021spatially], which could lead to the discovery of mechanisms representing cancer vulnerabilities resulting from interactions between the tumor and stroma cells.
3.  **Investigate cell-cell interactions:** From basic to applied tissue biology research, the study of cell-cell interactions is of high interest, especially the interactions that occur via ligand-receptor pairs. The construction of comprehensive databases of ligand-receptor interactions has been possible due the large amounts of single-cell data sets produced by researchers. A major contribution of ST to the study of tissue biology is the addition of the spatial context to previously identified ligand-receptor interactions. Because single-cell RNA-seq requires physical separation of cells, current ligand-receptor databases represent hypotheses which ST can help to address by using models of spatial co-localization, enabling in-situ examination of cell-cell interactions and communication [@raredon2023comprehensive; @wang2023promising].
4.  **Integrate imaging data:** Spatial transcriptomics data has enabled direct integration of gene expression measurements with digital images of the same (or adjacent) tissue. Improved molecular description and/or exploration of tissue niches or domains is now possible. One approach consists on differential expression of histopathology annotations done by an expert on tissue images (e.g., @ravi2022spatially). The oppoosite approach is possible, which uses unsupervised clustering of ST data assisted by color/intensity information derived from images. Machine learning for integration of ST and imaging data is an active area of development (e.g., @hu2021spagcn; @xu2022deepst; @tan2020spacell). Furthermore, ST data findings can be qualitatively validated by assessing the approximate location of regions such as immune-infiltrated areas or damaged tissue, often resulting from inspection of fluorescence microscopy.
5.  **Identify biomarkers and drug targets:** The use of ST allows the exploration of tissue niche-specific expression patterns and gene pathway analysis. This exploration can lead to generation of hypotheses about potential biomarkers for specific tissue functions or disease states. Furthermore, the molecular interactions predicted using scRNA-seq (e.g., ligand-receptor), can now be put in context of the larger tissue architecture using ST data. The spatial context of these interactions will likely boost the identification of novel drug targets, as well as improved understanding of current therapies [@lyubetskaya2022assessment; @zhang2022clinical].

## Overview of a spatial transcriptomics workflow

There is a large diversity in approaches to spatially profile tissues. Some ST technologies allow profiling at coarse cellular resolution, where regions of interest (ROIs) are usually identified by a pathologist. These ROIs may include tens of cells up to few hundreds (e.g., GeoMx @bergholtz2021best). Smaller ROI sizes can be found in other technologies such as Visium, where ROIs of 55uM of diameter (or "spots") often contain no more than 10 cells (<https://www.10xgenomics.com/resources/analysis-guides/integrating-single-cell-and-visium-spatial-gene-expression-data>). For finer cellular resolution, technologies such as MERFISH, SMI, or Xenium, among others, can measure gene expression at individual cells [@yue2023guidebook]. In general, there is a trade-off between the cellular resolution and molecular resolution, as the number of quantified genes and RNA molecules is lower in single-cell level spatial technologies compared to those at the ROI or spot level. In single-cell ST, often a panel of hundreds of genes is quantified, while in "mini-bulk" (ROI/spot) ST, it is possible to genes at the whole transcriptome level.

`FIGURE TRADEOFF CELLULAR AND MOLECULAR RESOLUTION`

In addition to the differences in cellular and molecular, there are fundamental differences in the chemistry used to count the RNA transcripts in the tissue [@wang2021spatial; @yue2023guidebook]. Capture or hybridization of RNA followed by sequencing, or fluorescent imaging are two of the most common techniques used in ST methods. Because of large diversity in resolution and chemical procedures among ST technologies, data collection workflows are equally diverse. Finally, each study poses specific questions that cannot be addressed with traditional scRNA-seq pipelines, requiring customized workflows.

Some of the commonalities in the workflows are presented here:

1.  **Sample preparation:** The preparation of a tissue sample will depend largely on the specific ST technology to be used. In general, this involves obtaining the tissue of interest in the form of a thin slice from a fresh frozen biopsy or a paraffin embedded tissue block. Tissue slices are generally about five to 10 micron of thickness. Given the instability of RNA molecules, the samples originating the tissue slices should be properly preserved and stabilized to maintain the integrity of RNA molecules. Many ST technologies are compatible with tissue microarrays (TMAs).

2.  **Capture or hybridization of RNA molecules:** In this step, the tissue sample is typically placed on a solid substrate, such as regular positively charged glass slides or vendor-designed slides. The latter category include spatially barcoded slides. (e.g., Visium [@staahl2016visualization] ), where RNA capture probes are contained in microscopic spots arranged in arrays or grids. The use of positively charged slides are used in technologies using *in-situ* sequencing or imaging-based methods, however, capture-based methods like GeoMx also employ this type of slide. Each method entails specific considerations. An example of these considerations include optimization of tissue permeabilization in Visium slides to release the RNA molecules. In the case of imaging-based methods, RNA molecules are hybridized with fluorescent probes that uniquely identify each RNA species [e.g., SMI [@he2022high], MERFISH [@zhang2021spatially] ].

3.  **RNA quantification:** The method used to count the number of captured or hybridized RNA molecules greatly varies from technology to technology. Capture methods often involve release of the RNA molecules from the tissue or slide, followed by library preparation, amplification, next generation sequencing, and read mapping to a reference genome. In this case, libraries are spatially multiplexed, whereby barcodes indicate the spatial location originating the captured RNA molecules. In imaging-based methods, segmentation is required to delineate the cell borders. Then, coded fluorescent probes are counted within each segmented cells.

4.  **Data quality control and pre-processing:** As with any omics technology, filtering and pre-processing is of paramount importance for downstream analysis. Spatial transcriuptomics data typically contain an exceess of zeroes and high gene dropout [@zhao2022modeling]. Removing genes expressed in very few spots or cells is often done. Similarly, it is advisable to remove spots with very few counts, however, care needs to exercided to not remove biological variation due to cellularity (i.e., areas with fewer cells tend to have less counts). Mitochondrial or ribosomal genes if available in the data, can be used to assess the level of tissue necrosis and filter accordingly [@ospina2023primer]. In imaging-based methods, the area of cells can be used to detect "doublets" generated during image segmentation. Once filtering has been performed, gene count normalization and transfromation is typically a part of pre-processing. Commonly used methods in scRNA-seq such as library-size normalization and log-transformation, are also commonplace in spatial transcriptomics studies. Methods that attempt technical effect correction such as SCTransform [@hafemeister2019normalization] can be also used.

5.  **Visualization:** Similar to scRNA-seq data, dimension reduction methods such as the Uniform Manifold Approximation and Projection (UMAP) are key to visualize the heterogeneity of the data set. Nonetheless, given the additional modality provided by the spatial coordinates, spatial gene expression heatmaps can be generated, whcih can be compared against the imaging data (e.g., H&, IHC, mIF) to gain further insights into overall tissue architecture.

6.  **Clustering and cell/tissue domain phenotyping:** There is a plethora of clustering approaches, ranging from employed in scRNA-seq analysis (e.g., Louvain) to novel neural network classification. Some methods take advantage of the spatial location information and/or tissue image to inform clustering. Compared to clustering, cell/domain phenotyping is an area of even more active development, withn the majority of methods relying on the use of a comprehensive single-cell, tissue specific atlas from which cell types (i.e., "labels") are obtained. Canonical marker-based phenotyping is still widely used, and in many cases unavoidable to identify specific cell populations. general, it is advisable to use the expert validation of a tissue biologist or pathologist to ascertain if clustering and phenotyping are capturing the tissue architecture adequately.

## Spatial transcriptomic data **strengths**:

-   **Preservation of Spatial Information:** Spatial transcriptomics allows the investigation of gene expression patterns within the context of tissue or cellular spatial organization. By preserving the spatial information of RNA molecules, it provides insights into how gene expression varies across different regions within a tissue sample.
-   **Identification of Cell Types and Subpopulations:** Spatial transcriptomics enables the identification and characterization of different cell types and subpopulations within a tissue sample based on their unique gene expression profiles. This information helps in understanding cellular heterogeneity and cell-cell interactions within the tissue.
-   **Visualization of Gene Expression Patterns:** By generating spatially resolved gene expression data, spatial transcriptomics allows for the visualization of gene expression patterns across the tissue sample. This visualization can be in the form of heatmaps, spatial expression plots, or spatially resolved gene expression atlases, providing a comprehensive view of gene expression within the tissue.
-   **Integration with Anatomical and Imaging Data:** Spatial transcriptomics data can be integrated with anatomical or imaging data, such as histological images or imaging modalities like fluorescence microscopy. This integration enhances the interpretation of spatial transcriptomic data by correlating gene expression patterns with tissue morphology or specific cellular structures.
-   **Discovery of Novel Cell-Cell Interactions and Signaling Pathways:** By examining gene expression profiles in the spatial context, spatial transcriptomics can uncover novel cell-cell interactions and signaling pathways within a tissue. It allows the identification of genes that are specifically expressed in close proximity to each other, revealing potential regulatory relationships and functional interactions between cells.
-   **Exploration of Spatially Regulated Biological Processes:** Spatial transcriptomics enables the investigation of spatially regulated biological processes, such as tissue gradients or developmental processes occurring in specific regions. It provides insights into spatially restricted gene expression patterns associated with tissue patterning, morphogenesis, or cellular differentiation.
-   **Hypothesis Generation and Biomarker Discovery:** Spatial transcriptomic analysis can generate hypotheses and identify potential biomarkers related to specific tissue functions or disease states. By linking gene expression patterns to tissue organization and pathology, spatial transcriptomics facilitates the discovery of spatially restricted gene signatures and potential diagnostic or prognostic markers.

## Spatial transcriptomic data **weaknesses**:

-   **Limited Spatial Resolution:** Spatial transcriptomic techniques have a finite spatial resolution, which determines the level of detail at which gene expression patterns can be analyzed. The resolution is influenced by factors such as the size of capture spots or the resolution of imaging techniques. Obtaining fine-grained spatial information may be challenging, especially in complex tissues or samples with high cellular density.
-   **Technical Variability and Experimental Artifacts:** Spatial transcriptomic analysis involves multiple experimental steps, including tissue processing, capture techniques, and sequencing. Each step introduces technical variability and potential experimental artifacts, which can impact the accuracy and reproducibility of the results. Controlling and minimizing these sources of variation is crucial but can be challenging.
-   **Limited Coverage of Transcripts:** Spatial transcriptomic methods often have limited coverage of the entire transcriptome. The capture techniques used may bias the representation of certain RNA molecules or favor highly expressed transcripts, potentially leading to a biased view of gene expression. Additionally, spatial transcriptomic methods may have limitations in capturing non-polyadenylated RNAs or low-abundance transcripts.
-   **Complex Data Analysis:** Analyzing spatial transcriptomic data requires advanced computational methods and expertise. Data analysis involves multiple steps, including image processing, spatial registration, normalization, and interpretation. The complexity of the data and the need for specialized bioinformatics tools and pipelines can pose challenges, particularly for researchers without extensive computational skills.
-   **Validation and Integration Challenges:** Spatial transcriptomic analysis generates hypotheses and provides spatially resolved gene expression information. However, validating the functional significance of identified gene expression patterns or cellular interactions may require additional experiments or techniques. Integrating spatial transcriptomic data with other omics data or imaging modalities can also be complex and may require careful data integration strategies.
-   **Cost and Time Considerations:** Spatial transcriptomic analysis can be relatively expensive and time-consuming compared to traditional transcriptomic techniques. The specialized protocols, reagents, and instrumentation required can add to the cost of the analysis. Moreover, the data generation and analysis processes can be time-intensive, which may limit the scalability of studies involving large sample sizes.
-   **Limited Accessibility of Techniques:** Some spatial transcriptomic techniques may have limited availability, accessibility, or may require specialized equipment or expertise. This can restrict their widespread use and accessibility to researchers who do not have access to the necessary resources or infrastructure.

## Tools for spatial transcriptomics

### Data processing:

#### Space Ranger:

-   **Pros:** spaceranger is a software package developed by 10x Genomics specifically for processing and analyzing spatial transcriptomic data generated by their platform. It provides a streamlined workflow for processing raw data, including image processing, spot calling, and counting transcripts. spaceranger offers integration with the Seurat R package, enabling downstream analysis using Seurat's extensive tools.
-   **Cons:** spaceranger is optimized for 10x Genomics data and may have limited compatibility with other spatial transcriptomic platforms. It may not offer the same level of flexibility as more general-purpose tools like Seurat or Scanpy.

#### GeomxTools

-   **Pros:**
-   **Cons:**

### Data exploration:

#### Seurat:

-   **Pros:** Seurat is a widely used R package that offers a comprehensive suite of tools for spatial transcriptomic analysis. It provides a variety of functions for data preprocessing, dimensionality reduction, clustering, and visualization. Seurat has a large user community, extensive documentation, and tutorials, making it accessible to researchers.
-   **Cons:** Seurat can be memory-intensive, particularly when working with large datasets. It may require familiarity with R programming and bioinformatics concepts for effective use. Additionally, Seurat's performance can depend on the specific computational resources available.

#### Scanpy:

-   **Pros:** Scanpy is a Python-based library specifically designed for single-cell and spatial transcriptomic analysis. It offers a range of functionalities for data preprocessing, clustering, trajectory analysis, and visualization. Scanpy is known for its scalability, efficiency, and flexibility. It integrates well with other Python libraries and frameworks, making it suitable for integration with other analysis pipelines.
-   **Cons:** Similar to Seurat, Scanpy may require some familiarity with Python programming and bioinformatics concepts. Users without prior programming experience may need to invest time in learning Python.

#### Giotto:

-   **Pros:** .
-   **Cons:** .

#### spatialGE:

-   **Pros:** .
-   **Cons:** .

#### Loupe:

-   **Pros:** .
-   **Cons:** .

#### ST Pipeline:

-   **Pros:** ST Pipeline is an open-source bioinformatics pipeline developed by the Spatial Transcriptomics consortium. It provides a complete workflow for spatial transcriptomic data analysis, including preprocessing, normalization, spot detection, and visualization. ST Pipeline supports various spatial transcriptomic platforms, making it versatile.
-   **Cons:** ST Pipeline may require some command-line proficiency and familiarity with Linux environments. Users may need to invest time in setting up the pipeline and configuring parameters based on their specific datasets and platforms.

### Clustering/tissue domain identification:

#### SpaGCN:

-   **Pros:** .
-   **Cons:** .

### Spatially variable gene identification:

#### SpatialDE:

-   **Pros:** SpatialDE is a Python package designed for detecting spatially variable genes from spatial transcriptomic data. It offers statistical methods to identify genes that exhibit spatial heterogeneity in expression across the tissue. SpatialDE provides a straightforward implementation for identifying spatially variable genes and their spatial patterns.
-   **Cons:** SpatialDE focuses on the identification of spatially variable genes and may not provide a comprehensive set of tools for the entire spatial transcriptomic analysis pipeline. Additional tools may be required for preprocessing, visualization, and downstream analysis.

### Deconvolution/phenotyping:

#### SPOTlight:

-   **Pros:** .
-   **Cons:** .

#### STdeconvolve:

-   **Pros:** .
-   **Cons:** .

#### InSituType:

-   **Pros:** .
-   **Cons:** .

#### SpatialDecon:

-   **Pros:** .
-   **Cons:** .

### Cell communication:

#### CellChat:

-   **Pros:** .
-   **Cons:** .

## More tools and tutorials regarding spatial transcriptomics

-   [Analysis, visualization, and integration of spatial datasets with Seurat](https://satijalab.org/seurat/articles/spatial_vignette.html)
-   [Sheffield Bioinformatics tutorial for spatial transcriptomics](https://github.com/sheffield-bioinformatics-core/spatial_transcriptomics_tutorial)
-   [Theis Lab SCOG workshop materials for spatial transcriptomics](https://github.com/theislab/spatial_scog_workshop_2022)
