
```{r, include = FALSE}
ottrpal::set_knitr_image_path()
```

# Microbiome Sequencing

<div class = "warning">
This chapter is incomplete! If you wish to contribute, please [go to this form](https://forms.gle/dqYgmKH8XXE2ohwD9) or our [GitHub page](https://github.com/fhdsl/Choosing_Genomics_Tools).
</div>

## Learning Objectives

```{r, fig.alt = "Learning Objectives", out.width = "100%", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g2668d07d0b9_0_0")
```

TODO: Why do we care about microbiome
## Goals of Amplicon analysis

## Data Collection and Processing of Amplicon Analysis
- QIIME 2 is a bioinformatics microbiome analysis platform. QIIME 2 has all the 
tools discussed in the course in one environment. 
    - There are currently 3 different interfaces: Python, Command Line Interface and Galaxy! 
    - add cool features of QIIME 2 

```{r, fig.alt = "Amplicon data collection and processing", out.width = "100%", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g2a06494e09c_0_0")
```

## Upstream Analysis
I have my data back from the sequencer, what do I do now?

TODO: Add Earth Microbiome Protocol (EMP) INFO


TODO: (I don't know if I like this section)


There are a couple of things to know about your data so you can do the proper 
upstream data preparation. 

1.  Is your sequence multiplex?

2. Does your sequence have the primers, adapter and barcode removed? If not, you will want to remove those in the quality filtering step?  


### Demultiplexing
Generally, there are two ways to get your data back from the sequencer. Multiplexed or demultiplexed. 
If your sequences are multiplexed, you will receive two or three files back. They will be a file for your barcodes, a file for your forward sequences, and if you have pair-end sequences, you will also have a file for your reverse sequences. 

Demultiplexing is a process where you separate the pooled sample files into a per sample basis, so instead of having a barcode, forward and reverse file, you have a forward and reverse file for each sample! 

Some Sequencers give you the data already demultiplexed and in that case you can skip this step! 

If your barcodes and primers are in your sequence demultiplexing becomes more complicated, however cutadapt has functionality that can help demultiplex. The documentation for demultiplexing with cutadapt is [here](https://cutadapt.readthedocs.io/en/stable/guide.html#demultiplexing). There is also a QIIME 2 implementation for paired-end data is located [here](https://docs.qiime2.org/2023.9/plugins/available/cutadapt/demux-paired/) and single-end data is located [here](https://docs.qiime2.org/2023.9/plugins/available/cutadapt/demux-single/). 

### Quality Filtering
Sequencing data is not perfect but there are some awesome methods that help us quality control our data. The method that you decide to use depends on what sequencer you used and whether or not your data has the barcodes/primers in it. 

#### Removing Adapters
If your data has barcodes or primers in the sequence you will want to remove those. Barcodes and Primers are synthetic DNA that were attached to help with selecting your region of interest, identifying samples and attaching to the sequencer, but not that the data is passed the sequencing part that genetic information is not useful,

Cutadapt is a method that is really effective at identifying your adapter sequence in high-throughput sequencing data. You can read more about cutadapt and the underlying methods [here](https://cutadapt.readthedocs.io/en/stable/). You can also find a QIIME 2 implementation of cutadapt, q2-cutadapt, [here](https://docs.qiime2.org/2023.9/plugins/available/cutadapt/#cutadapt). Cutadapt will trim out these adapters but it is not a replacement for quality filtering. 

#### Quality Filtering Steps 
Quality filtering encompasses a couple of methods to make sure the data is usable/.
1. checks sequence quality. 
2. denoises sequencing noise.
3. removes adapters, barcodes and primers (if applicable).
4. merges forward and reverse reads (if applicable).
5. checks for chimeras

Chimeras are when two separate sequences get tangled up and get sequenced. This results in a sequence in your data that has no biological meaning.

#### Defining Microbes 
There are two main ways for defining how sequences relate to the microbes in the microbiome: Amplicon Sequence Variants(ASVs) and Operational Taxonomic Units(OTUs). ASVs define an occurrence of a microbe as any occurrence of a unique sequence. OTUs cluster based on similarity, usually ranging from 97-99% similarity. OTUs define an occurrence of a microbe as a occurrence of any sequence in the similarity cluster. 

TODO: ask Greg, should I include OTU options in this? 
#### The Methods

ASV Quality Filtering Options: 

[Dada2](https://benjjneb.github.io/dada2/) is a method for de-replicating, filtering and merging that has a wide range of functionality. Dada2 also has a QIIME 2 documentation located [here](https://docs.qiime2.org/2023.9/plugins/available/dada2/#dada2). Dada2 can be used for [Pyro](https://docs.qiime2.org/2023.9/plugins/available/dada2/denoise-pyro/) and [Pacbio](https://docs.qiime2.org/2023.9/plugins/available/dada2/denoise-ccs/) sequencing as well as Illumina for [paired](https://docs.qiime2.org/2023.9/plugins/available/dada2/denoise-paired/) and [single](https://docs.qiime2.org/2023.9/plugins/available/dada2/denoise-single/) end reads. 

Another Method for quality filtering is [Deblur](https://github.com/biocore/deblur). Additionally the documentation for the QIIME 2 implementation can be found [here](https://docs.qiime2.org/2023.9/plugins/available/deblur/denoise-16S/). Deblur can only be used on 16s data because it uses a reference database that it aligns the data to in order to verify they are 16s data. Deblur is also only effective on data generated with Illumina sequencing. 


## Downstream Analysis  

### Phylogenetic Tree Construction

Many diversity metrics used in microbial ecology use phylogenetic relatedness to calculate diversity so we need to build a phylogenetic tree for those methods to reference. There are many ways to generate phylogenetic trees, however, because we only need these for diversity metrics and they are only using a very small amount of the genome, microbiome scientists tend to use quicker methods of phylogenetic construction.
There are two generally used methods. One involves creating a phylogenetic tree from scratch. This is typically done using [MAFFT](https://mafft.cbrc.jp/alignment/software/) to align sequences and calculate similarity. Other methods have a reference tree and insert the sequences into the reference tree, like using [SEPP](https://github.com/smirarab/sepp/tree/master). 

Additional QIIME 2 implementations for Phylogenetic Tree Construction:

- [fragment-insert sepp](https://docs.qiime2.org/2023.9/plugins/available/fragment-insertion/sepp/)

- [align-to-tree-mafft-fasttree](https://docs.qiime2.org/2023.9/plugins/available/phylogeny/align-to-tree-mafft-fasttree/)

- [align-to-tree-mafft-iqtree](https://docs.qiime2.org/2023.9/plugins/available/phylogeny/align-to-tree-mafft-iqtree/)

- [align-to-tree-mafft-raxml](https://docs.qiime2.org/2023.9/plugins/available/phylogeny/align-to-tree-mafft-raxml/)


### Rarefaction

An issue with microbiome data is that due to the nature of our sequencing technology, one sample might be 
more deeply sequenced compared to another sample. When samples are sequenced, it is not possible to guarantee that all the samples will be sequencing that same amount. This amount that a sample is sequences is referred to as **sequencing depth**. Varying sequencing depth can cause significant issues when trying to run diversity metrics or try to compare these samples. If you have samples with uneven sequencing depths, the diversity metrics that are calculated might indicate the sample that was sequencing deeper has more diversity simply because it was sequenced more. 

    For example, if I went to the desert and counted all the unique species of plants in a 100 mile radius and then I went to the rainforest and counted all the unique species in a 10 mile radius. I might find that they are more species in the desert simply because I investigated the desert more.

    Now, I have realized the fault in my study so I go back to desert and I count all the unique species of plants a 10 mile radius and I also go to the rainforest and counted all the unique species in a 10 mile radius.  In this version of the study I would probably find that the rainforest is the more diversity environment. 

    Eventhough I am investigating the desert less then I did in my first study, these environments are now comparable because I am investigating them equally.

Researchers typically solve this by choosing an even sampling depth for all samples. An **even sampling depth** a value you chose that you will randomly subset all your samples too. This way, just like in the example above, we are investigating each sample equally. The necessary evil of even sampling depths is that some samples might be lost because they do not meet the sampling depth minimum and that some samples that are over the selected sampling depth will loose features. 

They are two techniques that are used to apply even sampling depths. 

- **Rarefying**: a technique to remove uneven sequencing depth by subsampling without 
     replacement so that all samples have the same sampling
     depth. In QIIME2, Rarefying is currently done as part of the [core-metrics](https://docs.qiime2.org/2023.9/plugins/available/diversity/core-metrics/) and [core-metrics-phylogenetic](https://docs.qiime2.org/2023.9/plugins/available/diversity/core-metrics-phylogenetic/) pipelines, but can also be run[independently](https://docs.qiime2.org/2023.9/plugins/available/feature-table/rarefy/) of core-metrics. [alpha-rarefaction](https://docs.qiime2.org/2023.9/plugins/available/diversity/alpha-rarefaction/) and [beta-rarefaction](https://docs.qiime2.org/2023.9/plugins/available/diversity/beta-rarefaction/) can be used to confirm a reasonable sampling depth. 

- **Rarefaction**: an iterative technique to minimize effects of controlling for uneven 
     sampling depth where a feature table is rarefied multiple times (typically 100 or 1000), and diversity 
     metrics are applied to each rarefied table and subsequently used to compute summary statistics. TODO: Placeholder for q2-boots

See [Schloss (2024)](https://doi.org/10.1128/msphere.00355-23) for additional information on rarefying, rarefaction and controlling for an uneven sequencing depth.


### Alpha Diversity

### Beta Diversity

### Taxonomic Annotation
### Differential Abundance  
### Specialty Tools
#### Longitudinal Data 
#### Fecal Microbiota Transplant 
#### Predictive Models




## More resources

- [DNA methylation analysis with Galaxy tutorial](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/methylation-seq/tutorial.html)
- [The mint pipeline](https://github.com/sartorlab/mint/blob/master/README.md) for analyzing methylation and hydroxymethylation data.
- [Book chapter about finding methylation regions of interest](https://compgenomr.github.io/book/extracting-interesting-regions-differential-methylation-and-segmentation.html)
