<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 A Very General Genomics Overview | Choosing Genomics Tools</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 A Very General Genomics Overview | Choosing Genomics Tools" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 A Very General Genomics Overview | Choosing Genomics Tools" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="introduction.html"/>
<link rel="next" href="what-are-metadata.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#available-course-formats"><i class="fa fa-check"></i><b>0.1</b> Available course formats</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.1</b> Target Audience</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#topics-covered"><i class="fa fa-check"></i><b>1.2</b> Topics covered:</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.3</b> Motivation</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.4</b> Curriculum</a></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#how-to-use-the-course"><i class="fa fa-check"></i><b>1.5</b> How to use the course</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html"><i class="fa fa-check"></i><b>2</b> A Very General Genomics Overview</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#learning-objectives"><i class="fa fa-check"></i><b>2.1</b> Learning Objectives</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#what-do-genomics-workflows-look-like"><i class="fa fa-check"></i><b>2.1.1</b> What do genomics workflows look like?</a></li>
<li class="chapter" data-level="2.1.2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#basic-file-formats"><i class="fa fa-check"></i><b>2.1.2</b> Basic file formats</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#very-general-sequencing-workflow"><i class="fa fa-check"></i><b>2.2</b> Very General Sequencing Workflow</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#sequencing-file-formats"><i class="fa fa-check"></i><b>2.2.1</b> Sequencing file formats</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#very-general-microarray-workflow"><i class="fa fa-check"></i><b>2.3</b> Very General Microarray Workflow</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#microarray-file-formats"><i class="fa fa-check"></i><b>2.3.1</b> Microarray file formats</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#general-informatics-files"><i class="fa fa-check"></i><b>2.4</b> General informatics files</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#other-files"><i class="fa fa-check"></i><b>2.4.1</b> Other files</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="what-are-metadata.html"><a href="what-are-metadata.html"><i class="fa fa-check"></i><b>3</b> What are Metadata?</a>
<ul>
<li class="chapter" data-level="3.1" data-path="what-are-metadata.html"><a href="what-are-metadata.html#learning-objectives-1"><i class="fa fa-check"></i><b>3.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2" data-path="what-are-metadata.html"><a href="what-are-metadata.html#what-are-metadata-1"><i class="fa fa-check"></i><b>3.2</b> What are metadata?</a></li>
<li class="chapter" data-level="3.3" data-path="what-are-metadata.html"><a href="what-are-metadata.html#how-to-create-metadata"><i class="fa fa-check"></i><b>3.3</b> How to create metadata?</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="what-are-metadata.html"><a href="what-are-metadata.html#the-goals-in-creating-your-metadata"><i class="fa fa-check"></i><b>3.3.1</b> The goals in creating your metadata:</a></li>
<li class="chapter" data-level="3.3.2" data-path="what-are-metadata.html"><a href="what-are-metadata.html#to-recap"><i class="fa fa-check"></i><b>3.3.2</b> To recap:</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<a href="http://jhudatascience.org/"><img src="https://jhudatascience.org/images/dasl.png" style=" width: 80%; filter: grayscale(100%); padding-left: 40px; padding-top: 8px; vertical-align: top "</a>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Choosing Genomics Tools</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="a-very-general-genomics-overview" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> A Very General Genomics Overview</h1>
<div id="learning-objectives" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Learning Objectives</h2>
<p><img src="resources/images/02-genomics_overview_files/figure-html/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY_gd422c5de97_0_16.png" title="Learning objectives This chapter will demonstrate how to: Understand what will be covered in this course. Find information about your particular file format" alt="Learning objectives This chapter will demonstrate how to: Understand what will be covered in this course. Find information about your particular file format" width="480" /></p>
<p>In this chapter we are going to cover sequencing and microarray workflows at a very general high level overview to give you a first orientation. As we dive into specific data types and experiments, we will get into more specifics.
Here we will cover the most common file formats. If you have a file format you are dealing with that you don’t see listed here, it may be specific to your data type and we will discuss that more in that data type’s respective chapter. We still suggest you go through this chapter to give you a basic understanding of commonalities of all genomic data types and workflows</p>
<div id="what-do-genomics-workflows-look-like" class="section level3" number="2.1.1">
<h3><span class="header-section-number">2.1.1</span> What do genomics workflows look like?</h3>
<p>In the most general sense, all genomics data when originally collected is raw, it needs to undergo processing to be normalized and ready to use. Then normalized data is generally summarized in a way that is ready for it to be further consumed. Lastly, this summarized data is what can be used to make inferences and create plots and results tables.</p>
<p><img src="resources/images/02-genomics_overview_files/figure-html/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY_g12890ae15d7_0_20.png" title="In the most general sense, all genomics data when originally collected is raw, it needs to undergo processing to be normalized and ready to use. Then normalized data is generally summarized in a way that is ready for it to be further consumed. Lastly this summarized data is what can be used to make inferences and create plots and results tables. " alt="In the most general sense, all genomics data when originally collected is raw, it needs to undergo processing to be normalized and ready to use. Then normalized data is generally summarized in a way that is ready for it to be further consumed. Lastly this summarized data is what can be used to make inferences and create plots and results tables. " width="480" /></p>
</div>
<div id="basic-file-formats" class="section level3" number="2.1.2">
<h3><span class="header-section-number">2.1.2</span> Basic file formats</h3>
<p>Before we get into bioinformatic file types, we should establish some general file types that you likely have already worked with on your computer. These file types are used in all kinds of applications and not specific to bioinformatics.</p>
<div id="txt---text" class="section level4" number="2.1.2.1">
<h4><span class="header-section-number">2.1.2.1</span> TXT - Text</h4>
<p>A text file is a very basic file format that contains text!</p>
</div>
<div id="tsv---tab-separated-values" class="section level4" number="2.1.2.2">
<h4><span class="header-section-number">2.1.2.2</span> TSV - Tab Separated Values</h4>
<p>Tab separated values file is a text file is good for storing a data table.
It has rows and columns where each value is separated by (you guessed it), <em>tabs</em>.
Most commonly, if your genomics data has been provided to you in a TSV or CSV file, it has been processed and summarized! It will be your job to know <em>how</em> it was processed and summarized</p>
<p>Here the literal <code>⇥</code> represents tabs which often may show up invisible in your text editor’s preference settings.</p>
<pre><code>gene_id⇥sample_1⇥sample_2
gene_a⇥12⇥15,
gene_b⇥13⇥14</code></pre>
</div>
<div id="csv---comma-separated-values" class="section level4" number="2.1.2.3">
<h4><span class="header-section-number">2.1.2.3</span> CSV - Comma Separated Values</h4>
<p>A comma separated values file is list just like a TSV file but instead of values being separated by tabs it is separated by… (you guessed it), <em>commas</em>!</p>
<p>In its raw form, a CSV file might look like our example below (but if you open it with a program for spreadsheets, like Excel or Googlesheets, it will look like a table)</p>
<pre><code>gene_id, sample_1, sample_2,
gene_a, 12, 15,
gene_b, 13, 14</code></pre>
</div>
</div>
</div>
<div id="very-general-sequencing-workflow" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Very General Sequencing Workflow</h2>
<p>In the data type specific chapters, we will cover the sequencing data workflows and file formats in more detail. But in the most general sense, sequencing workflows look like this:</p>
<div id="04D1599E33D92E370515664A58E78BE1EEB_69816">
<div id="04D1599E33D92E370515664A58E78BE1EEB_69816_robot">
<a href="https://cloud.smartdraw.com/share.aspx/?pubDocShare=04D1599E33D92E370515664A58E78BE1EEB" target="_blank"><img src="https://cloud.smartdraw.com/cloudstorage/04D1599E33D92E370515664A58E78BE1EEB/preview2.png"></a>
</div>
</div>
<script src="https://cloud.smartdraw.com/plugins/html/js/sdjswidget_html.js" type="text/javascript"></script>
<script type="text/javascript">SDJS_Widget("04D1599E33D92E370515664A58E78BE1EEB",69816,1,"");</script>
<p><br/></p>
<div id="sequencing-file-formats" class="section level3" number="2.2.1">
<h3><span class="header-section-number">2.2.1</span> Sequencing file formats</h3>
<div id="sam---sequence-alignment-map" class="section level4" number="2.2.1.1">
<h4><span class="header-section-number">2.2.1.1</span> SAM - Sequence Alignment Map</h4>
<p>SAM Files are text based files that have sequence information. It generally has not been quantified or mapped. It is the reads in their raw form. <a href="https://samtools.github.io/hts-specs/SAMv1.pdf">For more about SAM files</a>.</p>
</div>
<div id="bam---binary-alignment-map" class="section level4" number="2.2.1.2">
<h4><span class="header-section-number">2.2.1.2</span> BAM - Binary Alignment Map</h4>
<p>BAM files are like SAM files but are compressed (made to take up less space on your computer). This means if you double click on a BAM file to look at it, it will look jumbled and unintelligible. You will need to convert it to a SAM file if you want to see it yourself (but this isn’t necessary necessarily).</p>
</div>
<div id="fasta---fast-a" class="section level4" number="2.2.1.3">
<h4><span class="header-section-number">2.2.1.3</span> FASTA - “fast A”</h4>
<p>Fasta files are sequence files that can be either nucleotide or amino acid sequences. They look something like this (the example below illustrating an amino acid sequence):</p>
<pre><code>&gt;SEQ_ID
GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT</code></pre>
<p>For <a href="https://en.wikipedia.org/wiki/FASTA_format">more about fasta files</a>.</p>
</div>
<div id="fastq---fast-q" class="section level4" number="2.2.1.4">
<h4><span class="header-section-number">2.2.1.4</span> FASTQ - “Fast q”</h4>
<p>A Fastq file is like a Fasta file except that it also contains information about the <strong>Q</strong>uality of the read. By quality, we mean, how sure was the sequencing machine that the nucleotide or amino acid called was indeed called correctly?</p>
<pre><code>@SEQ_ID
GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT
+
!&#39;&#39;*((((***+))%%%++)(%%%%).1***-+*&#39;&#39;))**55CCF&gt;&gt;&gt;&gt;&gt;&gt;CCCCCCC65</code></pre>
<p>For <a href="https://en.wikipedia.org/wiki/FASTQ_format">more about fastq files</a>.</p>
<p>Later in this course we will discuss the importance of examining the quality of your sequencing data and how to do that. If you received your data from a bioinformatics core it is possible that they’ve already done this quality analysis for you.</p>
<p><em>Sequencing data that is not of high enough quality should not be trusted!</em> It may need to be re-run entirely or may need extra processing (trimming) in order to make it more trustworthy. We will discuss this more in later chapters.</p>
</div>
<div id="bcl---binary-base-call-bcl-sequence-file-format" class="section level4" number="2.2.1.5">
<h4><span class="header-section-number">2.2.1.5</span> BCL - binary base call (BCL) sequence file format</h4>
<p>This type of sequence file is specific to Illumina data. In most cases, you will simply want to convert it to Fastq files for use with non-Illumina programs.</p>
<p><a href="https://medium.com/@marija190396/bcl-to-fastq-conversion-e289852823d0">More about BCL to Fastq conversion</a>.</p>
</div>
<div id="vcf---variant-call-format" class="section level4" number="2.2.1.6">
<h4><span class="header-section-number">2.2.1.6</span> VCF - Variant Call Format</h4>
<p>VCF files are further processed form of data than the sequence files we discussed above. VCF files are specially for storing only where a particular sample’s sequences differ or are <em>variant</em> from the reference genome or each other.</p>
<p>This will only be pertinent to you if you care about DNA variants. We will discuss this in the DNA seq chapter.</p>
<p>For <a href="https://en.wikipedia.org/wiki/Variant_Call_Format">more on VCF files</a>.</p>
</div>
<div id="maf---mutation-annotation-format" class="section level4" number="2.2.1.7">
<h4><span class="header-section-number">2.2.1.7</span> MAF - Mutation Annotation Format</h4>
<p>MAF files are aggregated versions of VCF files. So for a group of samples for which each has a VCF file, your entire group of samples’ variants will be summarized in the form of a MAF file.</p>
<p>For <a href="https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/#:~:text=Mutation%20Annotation%20Format%20(MAF)%20is,(or%20open%2Daccess).">more on MAF files</a>.</p>
</div>
</div>
</div>
<div id="very-general-microarray-workflow" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Very General Microarray Workflow</h2>
<p>In the data type specific chapters, we will cover the microarray workflow and file formats in more detail. But in the most general sense, microarray workflows look like this, note that the exact file formats are specific to the chip brand and type you use (e.g. Illumina, Affymetrix, Agilent, etc.):</p>
<div id="CB1322FC70C4A4B3A1BF2D8AC29A7E9FFF8_81516">
<div id="CB1322FC70C4A4B3A1BF2D8AC29A7E9FFF8_81516_robot">
<a href="https://cloud.smartdraw.com/share.aspx/?pubDocShare=CB1322FC70C4A4B3A1BF2D8AC29A7E9FFF8" target="_blank"><img src="https://cloud.smartdraw.com/cloudstorage/CB1322FC70C4A4B3A1BF2D8AC29A7E9FFF8/preview2.png"></a>
</div>
</div>
<script src="https://cloud.smartdraw.com/plugins/html/js/sdjswidget_html.js" type="text/javascript"></script>
<script type="text/javascript">SDJS_Widget("CB1322FC70C4A4B3A1BF2D8AC29A7E9FFF8",81516,1,"");</script>
<p><br/></p>
<div id="microarray-file-formats" class="section level3" number="2.3.1">
<h3><span class="header-section-number">2.3.1</span> Microarray file formats</h3>
<div id="idat---intensity-data-file" class="section level4" number="2.3.1.1">
<h4><span class="header-section-number">2.3.1.1</span> IDAT - intensity data file</h4>
<p>This is an Illumina microarray specific file that contains the chip image intensity information for each location on the microarray. It is a binary file, which means it will not be readable by double clicking and attempting to open the file directly.</p>
<p>Currently, Illumina appears to suggest directly converting IDAT files into a GTC format. We advise looking into <a href="https://github.com/freeseek/gtc2vcf">this package to help you do that</a>.</p>
<p><a href="https://www.illumina.com/content/dam/illumina-marketing/documents/products/technotes/technote_array_analysis_workflows.pdf">For more on IDAT files</a>.</p>
</div>
<div id="dat---data-file" class="section level4" number="2.3.1.2">
<h4><span class="header-section-number">2.3.1.2</span> DAT - data file</h4>
<p>This is an Affymetrix’ microarray specific file parallel to the IDAT file in that it contains the image intensity information for each location on the microarray. It’s stored as pixels.</p>
<p><a href="https://www.affymetrix.com/support/developer/powertools/changelog/gcos-agcc/dat.html">For more on DAT files</a>.</p>
</div>
<div id="cel" class="section level4" number="2.3.1.3">
<h4><span class="header-section-number">2.3.1.3</span> CEL</h4>
<p>This is an Affymetrix microarray specific file that is made from a DAT file but translated into numeric values. It is not normalized yet but can be normalized into a CHP file.</p>
<p><a href="https://www.affymetrix.com/support/developer/powertools/changelog/gcos-agcc/cel.html">For more on CEL files</a></p>
</div>
<div id="chp" class="section level4" number="2.3.1.4">
<h4><span class="header-section-number">2.3.1.4</span> CHP</h4>
<p>CHP files contain the gene-level and normalized data from an Affymetrix array chip. CHP files are obtained by normalizing and processing CEL files.</p>
<p><a href="https://www.affymetrix.com/support/developer/powertools/changelog/gcos-agcc/chp-xda.html">For more about CHP files</a>.</p>
</div>
</div>
</div>
<div id="general-informatics-files" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> General informatics files</h2>
<p>At various points in your genomics workflows, you may need to use other types of files to help you annotate your data. We’ll also discuss some of these common files that you may encounter:</p>
<div id="bed---browser-extensible-data" class="section level4" number="2.4.0.1">
<h4><span class="header-section-number">2.4.0.1</span> BED - Browser Extensible Data</h4>
<p>A BED file is a text file that has coordinates to genomic regions. THe other columns that accompany the genomic coordinates are variable depending on the context. But every BED file contains the <code>chrom</code>, <code>chromStart</code> and <code>chromEnd</code> columns to start.</p>
<p>A BED file might look like this:</p>
<pre><code>chrom   chromStart  chromEnd other_optional_columns
chr1    0      1000  good
chr2    100    3000  bad</code></pre>
<p>For <a href="https://en.wikipedia.org/wiki/BED_(file_format)">more on BED files</a>.</p>
</div>
<div id="gffgtf-general-feature-formatgene-transfer-format" class="section level4" number="2.4.0.2">
<h4><span class="header-section-number">2.4.0.2</span> GFF/GTF General Feature Format/Gene Transfer Format</h4>
<p>A GFF file is a tab delimited file that contains information about genomic features. These types of files are available from databases and what you can use to annotate your data.</p>
<p>You may see there are GFF2, GFF3, and GTF files. These only refer to different versions and variations. They generally have the same information. In general, GFF2 is being phased out so using GFF3 is generally a better bet unless the program or package you are using specifies it needs an older GFF2 version.</p>
<p>A GFF file may look like this (borrowed example from Ensembl):</p>
<pre><code>1 transcribed_unprocessed_pseudogene  gene        11869 14409 . + . gene_id &quot;ENSG00000223972&quot;; gene_name &quot;DDX11L1&quot;; gene_source &quot;havana&quot;; gene_biotype &quot;transcribed_unprocessed_pseudogene&quot;;</code></pre>
<p>Note that it will be useful for annotating genes and what we know about them.</p>
<p>For <a href="https://uswest.ensembl.org/info/website/upload/gff.html">more about GTF and GFF files</a>.</p>
</div>
<div id="other-files" class="section level3" number="2.4.1">
<h3><span class="header-section-number">2.4.1</span> Other files</h3>
<p>* If you didn’t see a file type listed you are looking for, take a look at this <a href="https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Data_formats">list by the BROAD</a>. Or, it may be covered in the data type specific chapters.</p>

</div>
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      <br>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="what-are-metadata.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
