<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 11 ATAC-Seq | Choosing Genomics Tools</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 11 ATAC-Seq | Choosing Genomics Tools" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 11 ATAC-Seq | Choosing Genomics Tools" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="single-cell-rna-seq.html"/>
<link rel="next" href="chip-seq.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#available-course-formats"><i class="fa fa-check"></i><b>0.1</b> Available course formats</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.1</b> Target Audience</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#topics-covered"><i class="fa fa-check"></i><b>1.2</b> Topics covered:</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.3</b> Motivation</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.4</b> Curriculum</a></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#how-to-use-the-course"><i class="fa fa-check"></i><b>1.5</b> How to use the course</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html"><i class="fa fa-check"></i><b>2</b> A Very General Genomics Overview</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#learning-objectives"><i class="fa fa-check"></i><b>2.1</b> Learning Objectives</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#what-do-genomics-workflows-look-like"><i class="fa fa-check"></i><b>2.1.1</b> What do genomics workflows look like?</a></li>
<li class="chapter" data-level="2.1.2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#basic-file-formats"><i class="fa fa-check"></i><b>2.1.2</b> Basic file formats</a></li>
<li class="chapter" data-level="2.1.3" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#sequencing-file-formats"><i class="fa fa-check"></i><b>2.1.3</b> Sequencing file formats</a></li>
<li class="chapter" data-level="2.1.4" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#microarray-file-formats"><i class="fa fa-check"></i><b>2.1.4</b> Microarray file formats</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#general-informatics-files"><i class="fa fa-check"></i><b>2.2</b> General informatics files</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#other-files"><i class="fa fa-check"></i><b>2.2.1</b> Other files</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="what-are-metadata.html"><a href="what-are-metadata.html"><i class="fa fa-check"></i><b>3</b> What are Metadata?</a>
<ul>
<li class="chapter" data-level="3.1" data-path="what-are-metadata.html"><a href="what-are-metadata.html#learning-objectives-1"><i class="fa fa-check"></i><b>3.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2" data-path="what-are-metadata.html"><a href="what-are-metadata.html#what-are-metadata-1"><i class="fa fa-check"></i><b>3.2</b> What are metadata?</a></li>
<li class="chapter" data-level="3.3" data-path="what-are-metadata.html"><a href="what-are-metadata.html#how-to-create-metadata"><i class="fa fa-check"></i><b>3.3</b> How to create metadata?</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="what-are-metadata.html"><a href="what-are-metadata.html#the-goals-in-creating-your-metadata"><i class="fa fa-check"></i><b>3.3.1</b> The goals in creating your metadata:</a></li>
<li class="chapter" data-level="3.3.2" data-path="what-are-metadata.html"><a href="what-are-metadata.html#to-recap"><i class="fa fa-check"></i><b>3.3.2</b> To recap:</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="microarray-data.html"><a href="microarray-data.html"><i class="fa fa-check"></i><b>4</b> Microarray Data</a>
<ul>
<li class="chapter" data-level="4.1" data-path="microarray-data.html"><a href="microarray-data.html#learning-objectives-2"><i class="fa fa-check"></i><b>4.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.2" data-path="microarray-data.html"><a href="microarray-data.html#summary-of-microarrays"><i class="fa fa-check"></i><b>4.2</b> Summary of microarrays</a></li>
<li class="chapter" data-level="4.3" data-path="microarray-data.html"><a href="microarray-data.html#how-do-microarrays-work"><i class="fa fa-check"></i><b>4.3</b> How do microarrays work?</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="microarray-data.html"><a href="microarray-data.html#pros"><i class="fa fa-check"></i><b>4.3.1</b> Pros:</a></li>
<li class="chapter" data-level="4.3.2" data-path="microarray-data.html"><a href="microarray-data.html#cons"><i class="fa fa-check"></i><b>4.3.2</b> Cons:</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="microarray-data.html"><a href="microarray-data.html#what-types-of-arrays-are-there"><i class="fa fa-check"></i><b>4.4</b> What types of arrays are there?</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="microarray-data.html"><a href="microarray-data.html#snp-arrays"><i class="fa fa-check"></i><b>4.4.1</b> SNP arrays</a></li>
<li class="chapter" data-level="4.4.2" data-path="microarray-data.html"><a href="microarray-data.html#gene-expression-arrays"><i class="fa fa-check"></i><b>4.4.2</b> Gene expression arrays</a></li>
<li class="chapter" data-level="4.4.3" data-path="microarray-data.html"><a href="microarray-data.html#dna-methylation-arrays"><i class="fa fa-check"></i><b>4.4.3</b> DNA methylation arrays</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="microarray-data.html"><a href="microarray-data.html#general-processing-of-microarray-data"><i class="fa fa-check"></i><b>4.5</b> General processing of microarray data</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="microarray-data.html"><a href="microarray-data.html#examples-2"><i class="fa fa-check"></i><b>4.5.1</b> Examples</a></li>
<li class="chapter" data-level="4.5.2" data-path="microarray-data.html"><a href="microarray-data.html#microarray-platforms"><i class="fa fa-check"></i><b>4.5.2</b> Microarray Platforms</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="microarray-data.html"><a href="microarray-data.html#very-general-microarray-workflow"><i class="fa fa-check"></i><b>4.6</b> Very General Microarray Workflow</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="microarray-data.html"><a href="microarray-data.html#microarray-file-formats-1"><i class="fa fa-check"></i><b>4.6.1</b> Microarray file formats</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="microarray-data.html"><a href="microarray-data.html#general-informatics-files-1"><i class="fa fa-check"></i><b>4.7</b> General informatics files</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="microarray-data.html"><a href="microarray-data.html#other-files-1"><i class="fa fa-check"></i><b>4.7.1</b> Other files</a></li>
<li class="chapter" data-level="4.7.2" data-path="microarray-data.html"><a href="microarray-data.html#microarray-processing-tutorials"><i class="fa fa-check"></i><b>4.7.2</b> Microarray processing tutorials:</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="sequencing-data.html"><a href="sequencing-data.html"><i class="fa fa-check"></i><b>5</b> Sequencing Data</a>
<ul>
<li class="chapter" data-level="5.1" data-path="sequencing-data.html"><a href="sequencing-data.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.2" data-path="sequencing-data.html"><a href="sequencing-data.html#how-does-sequencing-work"><i class="fa fa-check"></i><b>5.2</b> How does sequencing work?</a></li>
<li class="chapter" data-level="5.3" data-path="sequencing-data.html"><a href="sequencing-data.html#sequencing-concepts"><i class="fa fa-check"></i><b>5.3</b> Sequencing concepts</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="sequencing-data.html"><a href="sequencing-data.html#inherent-biases"><i class="fa fa-check"></i><b>5.3.1</b> Inherent biases</a></li>
<li class="chapter" data-level="5.3.2" data-path="sequencing-data.html"><a href="sequencing-data.html#pcr-amplification"><i class="fa fa-check"></i><b>5.3.2</b> PCR Amplification</a></li>
<li class="chapter" data-level="5.3.3" data-path="sequencing-data.html"><a href="sequencing-data.html#depth-of-coverage"><i class="fa fa-check"></i><b>5.3.3</b> Depth of coverage</a></li>
<li class="chapter" data-level="5.3.4" data-path="sequencing-data.html"><a href="sequencing-data.html#quality-controls"><i class="fa fa-check"></i><b>5.3.4</b> Quality controls</a></li>
<li class="chapter" data-level="5.3.5" data-path="sequencing-data.html"><a href="sequencing-data.html#alignment"><i class="fa fa-check"></i><b>5.3.5</b> Alignment</a></li>
<li class="chapter" data-level="5.3.6" data-path="sequencing-data.html"><a href="sequencing-data.html#single-end-vs-paired-end"><i class="fa fa-check"></i><b>5.3.6</b> Single End vs Paired End</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="sequencing-data.html"><a href="sequencing-data.html#very-general-sequencing-workflow"><i class="fa fa-check"></i><b>5.4</b> Very General Sequencing Workflow</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="sequencing-data.html"><a href="sequencing-data.html#sequencing-file-formats-1"><i class="fa fa-check"></i><b>5.4.1</b> Sequencing file formats</a></li>
<li class="chapter" data-level="5.4.2" data-path="sequencing-data.html"><a href="sequencing-data.html#other-files-2"><i class="fa fa-check"></i><b>5.4.2</b> Other files</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="dna-methods.html"><a href="dna-methods.html"><i class="fa fa-check"></i><b>6</b> DNA Methods</a>
<ul>
<li class="chapter" data-level="6.1" data-path="dna-methods.html"><a href="dna-methods.html#learning-objectives-4"><i class="fa fa-check"></i><b>6.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="6.2" data-path="dna-methods.html"><a href="dna-methods.html#what-are-the-goals-of-analyzing-dna-sequences"><i class="fa fa-check"></i><b>6.2</b> What are the goals of analyzing DNA sequences?</a></li>
<li class="chapter" data-level="6.3" data-path="dna-methods.html"><a href="dna-methods.html#comparison-of-dna-methods"><i class="fa fa-check"></i><b>6.3</b> Comparison of DNA methods</a></li>
<li class="chapter" data-level="6.4" data-path="dna-methods.html"><a href="dna-methods.html#how-to-choose-a-dna-sequencing-method"><i class="fa fa-check"></i><b>6.4</b> How to choose a DNA sequencing method</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="dna-methods.html"><a href="dna-methods.html#what-regions-of-the-genome-pertain-to-your-research-question"><i class="fa fa-check"></i><b>6.4.1</b> 1. What region(s) of the genome pertain to your research question?</a></li>
<li class="chapter" data-level="6.4.2" data-path="dna-methods.html"><a href="dna-methods.html#what-does-your-project-budget-allow-for"><i class="fa fa-check"></i><b>6.4.2</b> 2. What does your project budget allow for?</a></li>
<li class="chapter" data-level="6.4.3" data-path="dna-methods.html"><a href="dna-methods.html#what-is-your-detection-power-for-these-variants"><i class="fa fa-check"></i><b>6.4.3</b> 3. What is your detection power for these variants?</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="dna-methods.html"><a href="dna-methods.html#strengths-and-weaknesses-of-different-methods"><i class="fa fa-check"></i><b>6.5</b> Strengths and Weaknesses of different methods</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html"><i class="fa fa-check"></i><b>7</b> Whole Genome/Exome Sequencing Methods</a>
<ul>
<li class="chapter" data-level="7.1" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#learning-objectives-5"><i class="fa fa-check"></i><b>7.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="7.2" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#wgs-and-wgs-overview"><i class="fa fa-check"></i><b>7.2</b> WGS and WGS Overview</a></li>
<li class="chapter" data-level="7.3" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#advantages-and-disadvantages-of-wgs-vs-wxs"><i class="fa fa-check"></i><b>7.3</b> Advantages and Disadvantages of WGS vs WXS</a></li>
<li class="chapter" data-level="7.4" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#wgswxs-considerations"><i class="fa fa-check"></i><b>7.4</b> WGS/WXS Considerations</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#target-enrichment-techniques"><i class="fa fa-check"></i><b>7.4.1</b> Target enrichment techniques</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#dna-sequencing-pipeline-overview"><i class="fa fa-check"></i><b>7.5</b> DNA Sequencing Pipeline Overview</a></li>
<li class="chapter" data-level="7.6" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#data-pre-processing"><i class="fa fa-check"></i><b>7.6</b> Data Pre-processing</a></li>
<li class="chapter" data-level="7.7" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#commonly-used-tools"><i class="fa fa-check"></i><b>7.7</b> Commonly Used Tools</a></li>
<li class="chapter" data-level="7.8" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#data-pre-processing-tools"><i class="fa fa-check"></i><b>7.8</b> Data pre-processing tools</a></li>
<li class="chapter" data-level="7.9" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-somatic-and-germline-variant-identification"><i class="fa fa-check"></i><b>7.9</b> Tools for somatic and germline variant identification</a></li>
<li class="chapter" data-level="7.10" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-variant-calling-annotation"><i class="fa fa-check"></i><b>7.10</b> Tools for variant calling annotation</a></li>
<li class="chapter" data-level="7.11" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-copy-number-variation-analysis"><i class="fa fa-check"></i><b>7.11</b> Tools for copy number variation analysis</a></li>
<li class="chapter" data-level="7.12" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-data-visualization"><i class="fa fa-check"></i><b>7.12</b> Tools for data visualization</a></li>
<li class="chapter" data-level="7.13" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#resources-for-wgs"><i class="fa fa-check"></i><b>7.13</b> Resources for WGS</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="rna-methods.html"><a href="rna-methods.html"><i class="fa fa-check"></i><b>8</b> RNA Methods</a>
<ul>
<li class="chapter" data-level="8.1" data-path="rna-methods.html"><a href="rna-methods.html#learning-objectives-6"><i class="fa fa-check"></i><b>8.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="8.2" data-path="rna-methods.html"><a href="rna-methods.html#what-are-the-goals-of-gene-expression-analysis"><i class="fa fa-check"></i><b>8.2</b> What are the goals of gene expression analysis?</a></li>
<li class="chapter" data-level="8.3" data-path="rna-methods.html"><a href="rna-methods.html#comparison-of-rna-methods"><i class="fa fa-check"></i><b>8.3</b> Comparison of RNA methods</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html"><i class="fa fa-check"></i><b>9</b> Bulk RNA-seq</a>
<ul>
<li class="chapter" data-level="9.1" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#learning-objectives-7"><i class="fa fa-check"></i><b>9.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="9.2" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#where-rna-seq-data-comes-from"><i class="fa fa-check"></i><b>9.2</b> Where RNA-seq data comes from</a></li>
<li class="chapter" data-level="9.3" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-workflow"><i class="fa fa-check"></i><b>9.3</b> RNA-seq workflow</a></li>
<li class="chapter" data-level="9.4" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-data-strengths"><i class="fa fa-check"></i><b>9.4</b> RNA-seq data <strong>strengths</strong></a></li>
<li class="chapter" data-level="9.5" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-data-limitations"><i class="fa fa-check"></i><b>9.5</b> RNA-seq data <strong>limitations</strong></a></li>
<li class="chapter" data-level="9.6" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-data-considerations"><i class="fa fa-check"></i><b>9.6</b> RNA-seq data considerations</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#ribo-minus-vs-poly-a-selection"><i class="fa fa-check"></i><b>9.6.1</b> Ribo minus vs poly A selection</a></li>
<li class="chapter" data-level="9.6.2" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#transcriptome-mapping"><i class="fa fa-check"></i><b>9.6.2</b> Transcriptome mapping</a></li>
<li class="chapter" data-level="9.6.3" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#abundance-measures"><i class="fa fa-check"></i><b>9.6.3</b> Abundance measures</a></li>
<li class="chapter" data-level="9.6.4" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-downstream-analysis-tools"><i class="fa fa-check"></i><b>9.6.4</b> RNA-seq downstream analysis tools</a></li>
</ul></li>
<li class="chapter" data-level="9.7" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#more-reading-about-rna-seq-data"><i class="fa fa-check"></i><b>9.7</b> More reading about RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html"><i class="fa fa-check"></i><b>10</b> Single-cell RNA-seq</a>
<ul>
<li class="chapter" data-level="10.1" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#learning-objectives-8"><i class="fa fa-check"></i><b>10.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="10.2" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#where-single-cell-rna-seq-data-comes-from"><i class="fa fa-check"></i><b>10.2</b> Where single-cell RNA-seq data comes from</a></li>
<li class="chapter" data-level="10.3" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#single-cell-rna-seq-data-types"><i class="fa fa-check"></i><b>10.3</b> Single-cell RNA-seq data types</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#unique-molecular-identifiers"><i class="fa fa-check"></i><b>10.3.1</b> Unique Molecular identifiers</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#single-cell-rna-seq-tools"><i class="fa fa-check"></i><b>10.4</b> Single cell RNA-seq tools</a></li>
<li class="chapter" data-level="10.5" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#quantificationalignment"><i class="fa fa-check"></i><b>10.5</b> Quantification/Alignment</a></li>
<li class="chapter" data-level="10.6" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#quality-control"><i class="fa fa-check"></i><b>10.6</b> Quality control</a></li>
<li class="chapter" data-level="10.7" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#normalization-and-downstream-analyses"><i class="fa fa-check"></i><b>10.7</b> Normalization and downstream analyses</a></li>
<li class="chapter" data-level="10.8" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#useful-tutorials"><i class="fa fa-check"></i><b>10.8</b> Useful tutorials</a></li>
<li class="chapter" data-level="10.9" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#useful-readings"><i class="fa fa-check"></i><b>10.9</b> Useful readings</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="atac-seq.html"><a href="atac-seq.html"><i class="fa fa-check"></i><b>11</b> ATAC-Seq</a>
<ul>
<li class="chapter" data-level="11.1" data-path="atac-seq.html"><a href="atac-seq.html#learning-objectives-9"><i class="fa fa-check"></i><b>11.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="11.2" data-path="atac-seq.html"><a href="atac-seq.html#what-are-the-goals-of-atac-seq-analysis"><i class="fa fa-check"></i><b>11.2</b> What are the goals of ATAC-Seq analysis?</a></li>
<li class="chapter" data-level="11.3" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-general-workflow-overview"><i class="fa fa-check"></i><b>11.3</b> ATAC-Seq general workflow overview</a></li>
<li class="chapter" data-level="11.4" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-data-strengths"><i class="fa fa-check"></i><b>11.4</b> ATAC-Seq data <strong>strengths</strong>:</a></li>
<li class="chapter" data-level="11.5" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-data-limitations"><i class="fa fa-check"></i><b>11.5</b> ATAC-Seq data <strong>limitations</strong>:</a></li>
<li class="chapter" data-level="11.6" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-data-considerations"><i class="fa fa-check"></i><b>11.6</b> ATAC-Seq data considerations</a></li>
<li class="chapter" data-level="11.7" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-analysis-tools"><i class="fa fa-check"></i><b>11.7</b> ATAC-seq analysis tools</a></li>
<li class="chapter" data-level="11.8" data-path="atac-seq.html"><a href="atac-seq.html#more-resources-about-atac-seq-data"><i class="fa fa-check"></i><b>11.8</b> More resources about ATAC-seq data</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="chip-seq.html"><a href="chip-seq.html"><i class="fa fa-check"></i><b>12</b> ChIP-Seq</a>
<ul>
<li class="chapter" data-level="12.1" data-path="chip-seq.html"><a href="chip-seq.html#learning-objectives-10"><i class="fa fa-check"></i><b>12.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="12.2" data-path="chip-seq.html"><a href="chip-seq.html#what-are-the-goals-of-chip-seq-analysis"><i class="fa fa-check"></i><b>12.2</b> What are the goals of ChIP-Seq analysis?</a></li>
<li class="chapter" data-level="12.3" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-general-workflow-overview"><i class="fa fa-check"></i><b>12.3</b> ChIP-Seq general workflow overview</a></li>
<li class="chapter" data-level="12.4" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-data-strengths"><i class="fa fa-check"></i><b>12.4</b> ChIP-Seq data <strong>strengths</strong>:</a></li>
<li class="chapter" data-level="12.5" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-data-limitations"><i class="fa fa-check"></i><b>12.5</b> ChIP-Seq data <strong>limitations</strong>:</a></li>
<li class="chapter" data-level="12.6" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-data-considerations"><i class="fa fa-check"></i><b>12.6</b> ChIP-Seq data considerations</a></li>
<li class="chapter" data-level="12.7" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-analysis-tools"><i class="fa fa-check"></i><b>12.7</b> ChiP-seq analysis tools</a></li>
<li class="chapter" data-level="12.8" data-path="chip-seq.html"><a href="chip-seq.html#more-resources-about-chip-seq-data"><i class="fa fa-check"></i><b>12.8</b> More resources about ChiP-seq data</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html"><i class="fa fa-check"></i><b>13</b> DNA Methylation Sequencing Analysis Methods</a>
<ul>
<li class="chapter" data-level="13.1" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#learning-objectives-11"><i class="fa fa-check"></i><b>13.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="13.2" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#what-are-the-goals-of-analyzing-dna-methylation"><i class="fa fa-check"></i><b>13.2</b> What are the goals of analyzing DNA methylation?</a></li>
<li class="chapter" data-level="13.3" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-data-considerations"><i class="fa fa-check"></i><b>13.3</b> Methylation data considerations</a>
<ul>
<li class="chapter" data-level="13.3.1" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#beta-values-binomially-distributed"><i class="fa fa-check"></i><b>13.3.1</b> Beta values binomially distributed</a></li>
<li class="chapter" data-level="13.3.2" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#measuring-5mc-andor-5hmc"><i class="fa fa-check"></i><b>13.3.2</b> Measuring 5mC and/or 5hmC</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-data-workflow"><i class="fa fa-check"></i><b>13.4</b> Methylation data workflow</a></li>
<li class="chapter" data-level="13.5" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-tools"><i class="fa fa-check"></i><b>13.5</b> Methylation tools!</a>
<ul>
<li class="chapter" data-level="13.5.1" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#quality-control-1"><i class="fa fa-check"></i><b>13.5.1</b> Quality control</a></li>
<li class="chapter" data-level="13.5.2" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#genome-alignment"><i class="fa fa-check"></i><b>13.5.2</b> Genome Alignment</a></li>
<li class="chapter" data-level="13.5.3" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-calls"><i class="fa fa-check"></i><b>13.5.3</b> Methylation calls</a></li>
<li class="chapter" data-level="13.5.4" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#find-regions-of-interest"><i class="fa fa-check"></i><b>13.5.4</b> Find regions of interest!</a></li>
<li class="chapter" data-level="13.5.5" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#annotation-of-regions-of-interest"><i class="fa fa-check"></i><b>13.5.5</b> Annotation of regions of interest</a></li>
</ul></li>
<li class="chapter" data-level="13.6" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#more-resources"><i class="fa fa-check"></i><b>13.6</b> More resources</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html"><i class="fa fa-check"></i><b>14</b> General Data Analysis Tools</a>
<ul>
<li class="chapter" data-level="14.1" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#learning-objectives-12"><i class="fa fa-check"></i><b>14.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="14.2" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#command-line-vs-gui"><i class="fa fa-check"></i><b>14.2</b> Command Line vs GUI</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#bash"><i class="fa fa-check"></i><b>14.2.1</b> Bash</a></li>
<li class="chapter" data-level="14.2.2" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#r"><i class="fa fa-check"></i><b>14.2.2</b> R</a></li>
<li class="chapter" data-level="14.2.3" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#python"><i class="fa fa-check"></i><b>14.2.3</b> Python</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#data-visualization-tools"><i class="fa fa-check"></i><b>14.3</b> Data Visualization Tools</a></li>
<li class="chapter" data-level="14.4" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#more-resources-1"><i class="fa fa-check"></i><b>14.4</b> More resources</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="annotating-genomes.html"><a href="annotating-genomes.html"><i class="fa fa-check"></i><b>15</b> Annotating Genomes</a>
<ul>
<li class="chapter" data-level="15.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#learning-objectives-13"><i class="fa fa-check"></i><b>15.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="15.2" data-path="annotating-genomes.html"><a href="annotating-genomes.html#what-are-reference-genomes"><i class="fa fa-check"></i><b>15.2</b> What are reference genomes?</a></li>
<li class="chapter" data-level="15.3" data-path="annotating-genomes.html"><a href="annotating-genomes.html#what-are-genome-versions"><i class="fa fa-check"></i><b>15.3</b> What are genome versions?</a></li>
<li class="chapter" data-level="15.4" data-path="annotating-genomes.html"><a href="annotating-genomes.html#what-are-the-different-files"><i class="fa fa-check"></i><b>15.4</b> What are the different files?</a>
<ul>
<li class="chapter" data-level="15.4.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#how-to-download-annotation-files"><i class="fa fa-check"></i><b>15.4.1</b> How to download annotation files</a></li>
</ul></li>
<li class="chapter" data-level="15.5" data-path="annotating-genomes.html"><a href="annotating-genomes.html#considerations-for-annotating-genomic-data"><i class="fa fa-check"></i><b>15.5</b> Considerations for annotating genomic data</a>
<ul>
<li class="chapter" data-level="15.5.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#make-sure-you-have-the-right-file-to-start"><i class="fa fa-check"></i><b>15.5.1</b> Make sure you have the right file to start!</a></li>
<li class="chapter" data-level="15.5.2" data-path="annotating-genomes.html"><a href="annotating-genomes.html#be-consistent-in-your-annotations"><i class="fa fa-check"></i><b>15.5.2</b> Be consistent in your annotations</a></li>
<li class="chapter" data-level="15.5.3" data-path="annotating-genomes.html"><a href="annotating-genomes.html#be-clear-in-your-write-ups"><i class="fa fa-check"></i><b>15.5.3</b> Be clear in your write ups!</a></li>
</ul></li>
<li class="chapter" data-level="15.6" data-path="annotating-genomes.html"><a href="annotating-genomes.html#resources-you-will-need-for-annotation"><i class="fa fa-check"></i><b>15.6</b> Resources you will need for annotation!</a>
<ul>
<li class="chapter" data-level="15.6.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#annotation-databases"><i class="fa fa-check"></i><b>15.6.1</b> Annotation databases</a></li>
<li class="chapter" data-level="15.6.2" data-path="annotating-genomes.html"><a href="annotating-genomes.html#gui-based-annotation-tools"><i class="fa fa-check"></i><b>15.6.2</b> GUI based annotation tools</a></li>
<li class="chapter" data-level="15.6.3" data-path="annotating-genomes.html"><a href="annotating-genomes.html#command-line-based-tools"><i class="fa fa-check"></i><b>15.6.3</b> Command line based tools</a></li>
<li class="chapter" data-level="15.6.4" data-path="annotating-genomes.html"><a href="annotating-genomes.html#more-resources-about-genome-annotation"><i class="fa fa-check"></i><b>15.6.4</b> More resources about genome annotation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="https://hutchdatascience.org/"> The Fred Hutch Data Science Lab </a></p>
<a href="http://jhudatascience.org/"><img src="https://jhudatascience.org/images/dasl.png" style=" width: 80%; filter: grayscale(100%); padding-left: 40px; padding-top: 8px; vertical-align: top "</a>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Choosing Genomics Tools</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="atac-seq" class="section level1" number="11">
<h1><span class="header-section-number">Chapter 11</span> ATAC-Seq</h1>
<div id="learning-objectives-9" class="section level2" number="11.1">
<h2><span class="header-section-number">11.1</span> Learning Objectives</h2>
<p><img src="resources/images/08-ATAC-Seq_files/figure-html/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY_g12890ae15d7_0_66.png" title="Learning objectives This chapter will demonstrate how to: Understand the basics of ATAC-Seq data collection and processing workflow. Identify the next steps for your particular ATAC-Seq data. Formulate questions to ask about your ATAC-Seq data" alt="Learning objectives This chapter will demonstrate how to: Understand the basics of ATAC-Seq data collection and processing workflow. Identify the next steps for your particular ATAC-Seq data. Formulate questions to ask about your ATAC-Seq data" width="100%" /></p>
</div>
<div id="what-are-the-goals-of-atac-seq-analysis" class="section level2" number="11.2">
<h2><span class="header-section-number">11.2</span> What are the goals of ATAC-Seq analysis?</h2>
<p>The goals of ATAC-seq are to identify the accessible regions of the genome in a particular set of samples. These data allow us to understand the relationships between the chromatin accessibility patterns and cell states, and to understand the mechanistic causes and consequences of these chromatin accessibility patterns.</p>
<p>ATAC-seq data is generated by fragmenting the genome with the Tn5 endonuclease and sequencing the shorter DNA fragments. While most of the genome is associated with protein complexes that preclude the digestion of DNA by Tn5, some regions of the genome have accessible chromatin that can be cleaved by Tn5 resulting in short (&lt;500bp) fragments. These regions of the genome are of biological interest as they are likely to harbor transcription factor binding sites and to constitute cis-regulatory elements, genomic regions that are involved in the regulation of gene expression.</p>
<p><img src="resources/images/08-ATAC-Seq_files/figure-html/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY_g14492c87338_0_23.png" title="What does accessibility to chromatin represent? In ATAC-seq we are able to sequence open chromatin and find out DNA sequences where chromatin is accessible for activity. " alt="What does accessibility to chromatin represent? In ATAC-seq we are able to sequence open chromatin and find out DNA sequences where chromatin is accessible for activity. " width="100%" /></p>
</div>
<div id="atac-seq-general-workflow-overview" class="section level2" number="11.3">
<h2><span class="header-section-number">11.3</span> ATAC-Seq general workflow overview</h2>
<p>A basic ATAC-seq workflow involves mapping sequence reads to the genome, identifying peaks, assessing data quality, and identifying patterns of interest through clustering or identification of differentially accessible regions or other statistical means.</p>
<p>###Data quality metrics:</p>
<div id="number-of-mapped-reads" class="section level4" number="11.3.0.1">
<h4><span class="header-section-number">11.3.0.1</span> Number of mapped reads</h4>
<p>As for all DNA-sequencing based genomics technologies, a sufficient number of mapped reads is required to obtain meaningful results from a sample. You can read more about <a href="http://hutchdatascience.org/Choosing_Genomics_Tools/sequencing-data.html">general sequencing technologies in our previous chapter here</a>. For experiments on human samples this number should be greater than 20 million mapped unique reads.</p>
</div>
<div id="fragment-size-distribution" class="section level4" number="11.3.0.2">
<h4><span class="header-section-number">11.3.0.2</span> Fragment size distribution:</h4>
<p>ATAC-seq data is often generated using paired end sequencing technologies, which allow for characterization of ATAC-seq fragments. Histograms of these distributions using single base pair resolution bins reveal patterns of enrichment relative to the nucleosome scale of 147bp and the DNA-helix scale ~10.5bp.</p>
</div>
<div id="number-of-peaks" class="section level4" number="11.3.0.3">
<h4><span class="header-section-number">11.3.0.3</span> Number of peaks:</h4>
<p>Although the number of accessible chromatin regions can vary from one cell type to another, there are several regions that appear to be constitutively accessible across a most cell types. At least 20,000 peaks can be identified in a high quality experiment.</p>
</div>
<div id="frip-score-fraction-of-reads-in-peaks" class="section level4" number="11.3.0.4">
<h4><span class="header-section-number">11.3.0.4</span> FRiP score (fraction of reads in peaks)</h4>
<p>In high quality ATAC-seq data a large fraction of reads overlap with peaks, while in low quality data there is a high level of fragments that map to background regions.</p>
</div>
<div id="overlap-with-other-chromatin-accessibility-data" class="section level4" number="11.3.0.5">
<h4><span class="header-section-number">11.3.0.5</span> Overlap with other chromatin accessibility data</h4>
<p>Thousands of ATAC-seq samples have been produced in human and mouse. High quality ATAC-seq data will share a substantial proportion of peaks with many of these datasets.</p>
</div>
<div id="overlap-with-promoters" class="section level4" number="11.3.0.6">
<h4><span class="header-section-number">11.3.0.6</span> Overlap with promoters</h4>
<p>The promoter regions of many genes are constitutively accessible. Examining peak overlap with regions close to known protein coding gene transcription start sites can be used as a check for data quality.</p>
</div>
</div>
<div id="atac-seq-data-strengths" class="section level2" number="11.4">
<h2><span class="header-section-number">11.4</span> ATAC-Seq data <strong>strengths</strong>:</h2>
<ul>
<li>ATAC-seq is broadly used by many laboratories to characterize accessible chromatin in cell lines or sorted cells derived from tissues.</li>
<li>In principle, ATAC-seq can identify a large proportion of cis-regulatory elements.</li>
<li>In contrast to ChIP-seq, ATAC-seq does not require specific antibodies, or knowledge of the important transcription factors in a system.</li>
</ul>
</div>
<div id="atac-seq-data-limitations" class="section level2" number="11.5">
<h2><span class="header-section-number">11.5</span> ATAC-Seq data <strong>limitations</strong>:</h2>
<ul>
<li>ATAC-seq does not precisely identify the transcription factors or other chromatin associated factors that bind in or around chromatin accessible regions. This type of information needs to be inferred through analysis of transcription factor binding motif analysis or ChIP-seq data.</li>
<li>Accessible regions are not necessarily cis-regulatory regions, although many of them are.</li>
<li>The genes that are regulated by cis-regulatory elements cannot be identified conclusively by ATAC-seq alone.</li>
<li>ATAC-seq data can be biased making direct comparisons difficult.</li>
<li>ATAC-seq can be affected by batch effects.</li>
</ul>
</div>
<div id="atac-seq-data-considerations" class="section level2" number="11.6">
<h2><span class="header-section-number">11.6</span> ATAC-Seq data considerations</h2>
<p>The nucleosome is the fundamental unit of chromatin packaging in the genome and nucleosomal DNA is far less likely to be cleaved by the Tn5 nuclease than linker DNA. When DNA is fragmented by Tn5 the positions of the endpoints relative to the nucleosomes is an important consideration. When the ends are less than 147bp apart it is likely that both ends originate from the same linker region. Longer fragments can result from cuts on opposite sides of the same nucleosome, or even opposite sides of a genomic interval that encompasses multiple nucleosomes. The short fragments are therefore most likely to be nucleosome free and provide stronger evidence for transcription factor binding sites.</p>
<p>As will other genomics protocols, ATAC-seq data is subject to biases introduced in the ATAC-seq protocol and in the sequencing itself. Comparison of ATAC-seq data generated in different batches, by different laboratories or using different protocols might not be directly comparable. In addition, the Tn5 endonuclease does have biases in the precise DNA sequences it can cut. This should be taken into consideration when carrying out base pair resolution analyses including footprinting analysis and analysis of the effects of sequence variants on chromatin accessibility.</p>
<p>Read depth will impact ATAC-seq signal, but enzyme strength and conditions can also alter the distribution of cuts.</p>
<p>When using ATAC-seq data to answer biological questions it is important to understand what types of bias could impact the results. To ensure valid results the analysis needs to use appropriate statistical methods, ensure enough high quality ATAC-seq data is available, including controls, and possibly reframing the questions.</p>
</div>
<div id="atac-seq-analysis-tools" class="section level2" number="11.7">
<h2><span class="header-section-number">11.7</span> ATAC-seq analysis tools</h2>
<ul>
<li><a href="https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/atac-seq/tutorial.html">A Galaxy based tutorial for ATAC-seq</a> - Galaxy is a good recommendation for those new to informatics who would like a cloud-based GUI option to use for the analysis of their data.</li>
<li><a href="https://macs3-project.github.io/MACS/">MACS - Model-based analysis for ChIP-Seq</a> - A command line tool for the identification of transcription factor binding sites. Can be used with ChIP-seq or ATAC-seq.</li>
<li><a href="https://github.com/liulab-dfci/CHIPS">CHIPS</a> - A Snakemake pipeline for quality control and reproducible processing of chromatin profiling data. This tool will require some snakemake and coding knowledge. For more recommendations about coding see our later chapter about <a href="">general data analysis tools</a>.</li>
<li><a href="http://cistrome.org/db/#/">Cistrome DB</a> - a visual tool to allow you to browse your ATAC-seq data.</li>
<li><a href="https://github.com/zang-lab/SELMA">SELMA - Simplex Encoded Linear Model for Accessible Chromatin</a> - SELMA is a python based tool for the assessment of biases in Chromatin based data.</li>
</ul>
</div>
<div id="more-resources-about-atac-seq-data" class="section level2" number="11.8">
<h2><span class="header-section-number">11.8</span> More resources about ATAC-seq data</h2>
<ul>
<li><a href="https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/atac-seq/slides.html#1">ATAC-seq overview from Galaxy</a> - these slides explain the overarching concepts of ATAC-seq.</li>
<li><a href="https://informatics.fas.harvard.edu/atac-seq-guidelines.html">ATAC seq guidelines from Harvard</a> - this workflow runs through step by step how to analysis ATAC-seq data and what different parameters mean.</li>
<li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1929-3">ATAC-seq review</a> - this paper gives a great overview of ATAC-seq data and step by step what needs to be considered.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4473780/">Identifying and mitigating bias in chromatin</a></li>
<li><a href="https://f1000research.com/articles/10-517">CHIP Snakemake pipeline for analyzing ChIP-seq and chromatin accessibility data</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4018771/">Paper on bias in DNase-seq footprinting analysis and fragment size effects, similar comments apply to ATAC-seq</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9492688/">SELMA Method for evaluating footprint bias in ATAC-seq</a></li>
</ul>

</div>
</div>
<hr>
<center> 
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      <br>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="single-cell-rna-seq.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="chip-seq.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
