<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 16 Genomic Tool Glossary | Choosing Genomics Tools</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 16 Genomic Tool Glossary | Choosing Genomics Tools" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 16 Genomic Tool Glossary | Choosing Genomics Tools" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/ITN_favicon.ico" type="image/x-icon" />
<link rel="prev" href="annotating-genomes.html"/>
<link rel="next" href="about-the-authors.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="assets/style_ITN.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.itcrtraining.org/"><img src="assets/ITN_logo.png" style="padding-left: 15px; padding-top: 8px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#available-course-formats"><i class="fa fa-check"></i><b>0.1</b> Available course formats</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#target-audience"><i class="fa fa-check"></i><b>1.1</b> Target Audience</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#topics-covered"><i class="fa fa-check"></i><b>1.2</b> Topics covered:</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.3</b> Motivation</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#curriculum"><i class="fa fa-check"></i><b>1.4</b> Curriculum</a></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#how-to-use-the-course"><i class="fa fa-check"></i><b>1.5</b> How to use the course</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html"><i class="fa fa-check"></i><b>2</b> A Very General Genomics Overview</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#learning-objectives"><i class="fa fa-check"></i><b>2.1</b> Learning Objectives</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#what-do-genomics-workflows-look-like"><i class="fa fa-check"></i><b>2.1.1</b> What do genomics workflows look like?</a></li>
<li class="chapter" data-level="2.1.2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#basic-file-formats"><i class="fa fa-check"></i><b>2.1.2</b> Basic file formats</a></li>
<li class="chapter" data-level="2.1.3" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#sequencing-file-formats"><i class="fa fa-check"></i><b>2.1.3</b> Sequencing file formats</a></li>
<li class="chapter" data-level="2.1.4" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#microarray-file-formats"><i class="fa fa-check"></i><b>2.1.4</b> Microarray file formats</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#general-informatics-files"><i class="fa fa-check"></i><b>2.2</b> General informatics files</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="a-very-general-genomics-overview.html"><a href="a-very-general-genomics-overview.html#other-files"><i class="fa fa-check"></i><b>2.2.1</b> Other files</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="what-are-metadata.html"><a href="what-are-metadata.html"><i class="fa fa-check"></i><b>3</b> What are Metadata?</a>
<ul>
<li class="chapter" data-level="3.1" data-path="what-are-metadata.html"><a href="what-are-metadata.html#learning-objectives-1"><i class="fa fa-check"></i><b>3.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2" data-path="what-are-metadata.html"><a href="what-are-metadata.html#what-are-metadata-1"><i class="fa fa-check"></i><b>3.2</b> What are metadata?</a></li>
<li class="chapter" data-level="3.3" data-path="what-are-metadata.html"><a href="what-are-metadata.html#how-to-create-metadata"><i class="fa fa-check"></i><b>3.3</b> How to create metadata?</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="what-are-metadata.html"><a href="what-are-metadata.html#the-goals-in-creating-your-metadata"><i class="fa fa-check"></i><b>3.3.1</b> The goals in creating your metadata:</a></li>
<li class="chapter" data-level="3.3.2" data-path="what-are-metadata.html"><a href="what-are-metadata.html#to-recap"><i class="fa fa-check"></i><b>3.3.2</b> To recap:</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="microarray-data.html"><a href="microarray-data.html"><i class="fa fa-check"></i><b>4</b> Microarray Data</a>
<ul>
<li class="chapter" data-level="4.1" data-path="microarray-data.html"><a href="microarray-data.html#learning-objectives-2"><i class="fa fa-check"></i><b>4.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.2" data-path="microarray-data.html"><a href="microarray-data.html#summary-of-microarrays"><i class="fa fa-check"></i><b>4.2</b> Summary of microarrays</a></li>
<li class="chapter" data-level="4.3" data-path="microarray-data.html"><a href="microarray-data.html#how-do-microarrays-work"><i class="fa fa-check"></i><b>4.3</b> How do microarrays work?</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="microarray-data.html"><a href="microarray-data.html#pros"><i class="fa fa-check"></i><b>4.3.1</b> Pros:</a></li>
<li class="chapter" data-level="4.3.2" data-path="microarray-data.html"><a href="microarray-data.html#cons"><i class="fa fa-check"></i><b>4.3.2</b> Cons:</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="microarray-data.html"><a href="microarray-data.html#what-types-of-arrays-are-there"><i class="fa fa-check"></i><b>4.4</b> What types of arrays are there?</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="microarray-data.html"><a href="microarray-data.html#snp-arrays"><i class="fa fa-check"></i><b>4.4.1</b> SNP arrays</a></li>
<li class="chapter" data-level="4.4.2" data-path="microarray-data.html"><a href="microarray-data.html#gene-expression-arrays"><i class="fa fa-check"></i><b>4.4.2</b> Gene expression arrays</a></li>
<li class="chapter" data-level="4.4.3" data-path="microarray-data.html"><a href="microarray-data.html#dna-methylation-arrays"><i class="fa fa-check"></i><b>4.4.3</b> DNA methylation arrays</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="microarray-data.html"><a href="microarray-data.html#general-processing-of-microarray-data"><i class="fa fa-check"></i><b>4.5</b> General processing of microarray data</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="microarray-data.html"><a href="microarray-data.html#examples-2"><i class="fa fa-check"></i><b>4.5.1</b> Examples</a></li>
<li class="chapter" data-level="4.5.2" data-path="microarray-data.html"><a href="microarray-data.html#microarray-platforms"><i class="fa fa-check"></i><b>4.5.2</b> Microarray Platforms</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="microarray-data.html"><a href="microarray-data.html#very-general-microarray-workflow"><i class="fa fa-check"></i><b>4.6</b> Very General Microarray Workflow</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="microarray-data.html"><a href="microarray-data.html#microarray-file-formats-1"><i class="fa fa-check"></i><b>4.6.1</b> Microarray file formats</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="microarray-data.html"><a href="microarray-data.html#general-informatics-files-1"><i class="fa fa-check"></i><b>4.7</b> General informatics files</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="microarray-data.html"><a href="microarray-data.html#other-files-1"><i class="fa fa-check"></i><b>4.7.1</b> Other files</a></li>
<li class="chapter" data-level="4.7.2" data-path="microarray-data.html"><a href="microarray-data.html#microarray-processing-tutorials"><i class="fa fa-check"></i><b>4.7.2</b> Microarray processing tutorials:</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="sequencing-data.html"><a href="sequencing-data.html"><i class="fa fa-check"></i><b>5</b> Sequencing Data</a>
<ul>
<li class="chapter" data-level="5.1" data-path="sequencing-data.html"><a href="sequencing-data.html#learning-objectives-3"><i class="fa fa-check"></i><b>5.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.2" data-path="sequencing-data.html"><a href="sequencing-data.html#how-does-sequencing-work"><i class="fa fa-check"></i><b>5.2</b> How does sequencing work?</a></li>
<li class="chapter" data-level="5.3" data-path="sequencing-data.html"><a href="sequencing-data.html#sequencing-concepts"><i class="fa fa-check"></i><b>5.3</b> Sequencing concepts</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="sequencing-data.html"><a href="sequencing-data.html#inherent-biases"><i class="fa fa-check"></i><b>5.3.1</b> Inherent biases</a></li>
<li class="chapter" data-level="5.3.2" data-path="sequencing-data.html"><a href="sequencing-data.html#pcr-amplification"><i class="fa fa-check"></i><b>5.3.2</b> PCR Amplification</a></li>
<li class="chapter" data-level="5.3.3" data-path="sequencing-data.html"><a href="sequencing-data.html#depth-of-coverage"><i class="fa fa-check"></i><b>5.3.3</b> Depth of coverage</a></li>
<li class="chapter" data-level="5.3.4" data-path="sequencing-data.html"><a href="sequencing-data.html#quality-controls"><i class="fa fa-check"></i><b>5.3.4</b> Quality controls</a></li>
<li class="chapter" data-level="5.3.5" data-path="sequencing-data.html"><a href="sequencing-data.html#alignment"><i class="fa fa-check"></i><b>5.3.5</b> Alignment</a></li>
<li class="chapter" data-level="5.3.6" data-path="sequencing-data.html"><a href="sequencing-data.html#single-end-vs-paired-end"><i class="fa fa-check"></i><b>5.3.6</b> Single End vs Paired End</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="sequencing-data.html"><a href="sequencing-data.html#very-general-sequencing-workflow"><i class="fa fa-check"></i><b>5.4</b> Very General Sequencing Workflow</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="sequencing-data.html"><a href="sequencing-data.html#sequencing-file-formats-1"><i class="fa fa-check"></i><b>5.4.1</b> Sequencing file formats</a></li>
<li class="chapter" data-level="5.4.2" data-path="sequencing-data.html"><a href="sequencing-data.html#other-files-2"><i class="fa fa-check"></i><b>5.4.2</b> Other files</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="dna-methods.html"><a href="dna-methods.html"><i class="fa fa-check"></i><b>6</b> DNA Methods</a>
<ul>
<li class="chapter" data-level="6.1" data-path="dna-methods.html"><a href="dna-methods.html#learning-objectives-4"><i class="fa fa-check"></i><b>6.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="6.2" data-path="dna-methods.html"><a href="dna-methods.html#what-are-the-goals-of-analyzing-dna-sequences"><i class="fa fa-check"></i><b>6.2</b> What are the goals of analyzing DNA sequences?</a></li>
<li class="chapter" data-level="6.3" data-path="dna-methods.html"><a href="dna-methods.html#comparison-of-dna-methods"><i class="fa fa-check"></i><b>6.3</b> Comparison of DNA methods</a></li>
<li class="chapter" data-level="6.4" data-path="dna-methods.html"><a href="dna-methods.html#how-to-choose-a-dna-sequencing-method"><i class="fa fa-check"></i><b>6.4</b> How to choose a DNA sequencing method</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="dna-methods.html"><a href="dna-methods.html#what-regions-of-the-genome-pertain-to-your-research-question"><i class="fa fa-check"></i><b>6.4.1</b> 1. What region(s) of the genome pertain to your research question?</a></li>
<li class="chapter" data-level="6.4.2" data-path="dna-methods.html"><a href="dna-methods.html#what-does-your-project-budget-allow-for"><i class="fa fa-check"></i><b>6.4.2</b> 2. What does your project budget allow for?</a></li>
<li class="chapter" data-level="6.4.3" data-path="dna-methods.html"><a href="dna-methods.html#what-is-your-detection-power-for-these-variants"><i class="fa fa-check"></i><b>6.4.3</b> 3. What is your detection power for these variants?</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="dna-methods.html"><a href="dna-methods.html#strengths-and-weaknesses-of-different-methods"><i class="fa fa-check"></i><b>6.5</b> Strengths and Weaknesses of different methods</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html"><i class="fa fa-check"></i><b>7</b> Whole Genome/Exome Sequencing Methods</a>
<ul>
<li class="chapter" data-level="7.1" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#learning-objectives-5"><i class="fa fa-check"></i><b>7.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="7.2" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#wgs-and-wgs-overview"><i class="fa fa-check"></i><b>7.2</b> WGS and WGS Overview</a></li>
<li class="chapter" data-level="7.3" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#advantages-and-disadvantages-of-wgs-vs-wxs"><i class="fa fa-check"></i><b>7.3</b> Advantages and Disadvantages of WGS vs WXS</a></li>
<li class="chapter" data-level="7.4" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#wgswxs-considerations"><i class="fa fa-check"></i><b>7.4</b> WGS/WXS Considerations</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#target-enrichment-techniques"><i class="fa fa-check"></i><b>7.4.1</b> Target enrichment techniques</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#dna-sequencing-pipeline-overview"><i class="fa fa-check"></i><b>7.5</b> DNA Sequencing Pipeline Overview</a></li>
<li class="chapter" data-level="7.6" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#data-pre-processing"><i class="fa fa-check"></i><b>7.6</b> Data Pre-processing</a></li>
<li class="chapter" data-level="7.7" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#commonly-used-tools"><i class="fa fa-check"></i><b>7.7</b> Commonly Used Tools</a></li>
<li class="chapter" data-level="7.8" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#data-pre-processing-tools"><i class="fa fa-check"></i><b>7.8</b> Data pre-processing tools</a></li>
<li class="chapter" data-level="7.9" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-somatic-and-germline-variant-identification"><i class="fa fa-check"></i><b>7.9</b> Tools for somatic and germline variant identification</a></li>
<li class="chapter" data-level="7.10" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-variant-calling-annotation"><i class="fa fa-check"></i><b>7.10</b> Tools for variant calling annotation</a></li>
<li class="chapter" data-level="7.11" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-copy-number-variation-analysis"><i class="fa fa-check"></i><b>7.11</b> Tools for copy number variation analysis</a></li>
<li class="chapter" data-level="7.12" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#tools-for-data-visualization"><i class="fa fa-check"></i><b>7.12</b> Tools for data visualization</a></li>
<li class="chapter" data-level="7.13" data-path="whole-genomeexome-sequencing-methods.html"><a href="whole-genomeexome-sequencing-methods.html#resources-for-wgs"><i class="fa fa-check"></i><b>7.13</b> Resources for WGS</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="rna-methods.html"><a href="rna-methods.html"><i class="fa fa-check"></i><b>8</b> RNA Methods</a>
<ul>
<li class="chapter" data-level="8.1" data-path="rna-methods.html"><a href="rna-methods.html#learning-objectives-6"><i class="fa fa-check"></i><b>8.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="8.2" data-path="rna-methods.html"><a href="rna-methods.html#what-are-the-goals-of-gene-expression-analysis"><i class="fa fa-check"></i><b>8.2</b> What are the goals of gene expression analysis?</a></li>
<li class="chapter" data-level="8.3" data-path="rna-methods.html"><a href="rna-methods.html#comparison-of-rna-methods"><i class="fa fa-check"></i><b>8.3</b> Comparison of RNA methods</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html"><i class="fa fa-check"></i><b>9</b> Bulk RNA-seq</a>
<ul>
<li class="chapter" data-level="9.1" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#learning-objectives-7"><i class="fa fa-check"></i><b>9.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="9.2" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#where-rna-seq-data-comes-from"><i class="fa fa-check"></i><b>9.2</b> Where RNA-seq data comes from</a></li>
<li class="chapter" data-level="9.3" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-workflow"><i class="fa fa-check"></i><b>9.3</b> RNA-seq workflow</a></li>
<li class="chapter" data-level="9.4" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-data-strengths"><i class="fa fa-check"></i><b>9.4</b> RNA-seq data <strong>strengths</strong></a></li>
<li class="chapter" data-level="9.5" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-data-limitations"><i class="fa fa-check"></i><b>9.5</b> RNA-seq data <strong>limitations</strong></a></li>
<li class="chapter" data-level="9.6" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-data-considerations"><i class="fa fa-check"></i><b>9.6</b> RNA-seq data considerations</a>
<ul>
<li class="chapter" data-level="9.6.1" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#ribo-minus-vs-poly-a-selection"><i class="fa fa-check"></i><b>9.6.1</b> Ribo minus vs poly A selection</a></li>
<li class="chapter" data-level="9.6.2" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#transcriptome-mapping"><i class="fa fa-check"></i><b>9.6.2</b> Transcriptome mapping</a></li>
<li class="chapter" data-level="9.6.3" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#abundance-measures"><i class="fa fa-check"></i><b>9.6.3</b> Abundance measures</a></li>
<li class="chapter" data-level="9.6.4" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#rna-seq-downstream-analysis-tools"><i class="fa fa-check"></i><b>9.6.4</b> RNA-seq downstream analysis tools</a></li>
</ul></li>
<li class="chapter" data-level="9.7" data-path="bulk-rna-seq.html"><a href="bulk-rna-seq.html#more-reading-about-rna-seq-data"><i class="fa fa-check"></i><b>9.7</b> More reading about RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html"><i class="fa fa-check"></i><b>10</b> Single-cell RNA-seq</a>
<ul>
<li class="chapter" data-level="10.1" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#learning-objectives-8"><i class="fa fa-check"></i><b>10.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="10.2" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#where-single-cell-rna-seq-data-comes-from"><i class="fa fa-check"></i><b>10.2</b> Where single-cell RNA-seq data comes from</a></li>
<li class="chapter" data-level="10.3" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#single-cell-rna-seq-data-types"><i class="fa fa-check"></i><b>10.3</b> Single-cell RNA-seq data types</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#unique-molecular-identifiers"><i class="fa fa-check"></i><b>10.3.1</b> Unique Molecular identifiers</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#single-cell-rna-seq-tools"><i class="fa fa-check"></i><b>10.4</b> Single cell RNA-seq tools</a></li>
<li class="chapter" data-level="10.5" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#quantificationalignment"><i class="fa fa-check"></i><b>10.5</b> Quantification/Alignment</a></li>
<li class="chapter" data-level="10.6" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#quality-control"><i class="fa fa-check"></i><b>10.6</b> Quality control</a></li>
<li class="chapter" data-level="10.7" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#normalization-and-downstream-analyses"><i class="fa fa-check"></i><b>10.7</b> Normalization and downstream analyses</a></li>
<li class="chapter" data-level="10.8" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#useful-tutorials"><i class="fa fa-check"></i><b>10.8</b> Useful tutorials</a></li>
<li class="chapter" data-level="10.9" data-path="single-cell-rna-seq.html"><a href="single-cell-rna-seq.html#useful-readings"><i class="fa fa-check"></i><b>10.9</b> Useful readings</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="atac-seq.html"><a href="atac-seq.html"><i class="fa fa-check"></i><b>11</b> ATAC-Seq</a>
<ul>
<li class="chapter" data-level="11.1" data-path="atac-seq.html"><a href="atac-seq.html#learning-objectives-9"><i class="fa fa-check"></i><b>11.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="11.2" data-path="atac-seq.html"><a href="atac-seq.html#what-are-the-goals-of-atac-seq-analysis"><i class="fa fa-check"></i><b>11.2</b> What are the goals of ATAC-Seq analysis?</a></li>
<li class="chapter" data-level="11.3" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-general-workflow-overview"><i class="fa fa-check"></i><b>11.3</b> ATAC-Seq general workflow overview</a></li>
<li class="chapter" data-level="11.4" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-data-strengths"><i class="fa fa-check"></i><b>11.4</b> ATAC-Seq data <strong>strengths</strong>:</a></li>
<li class="chapter" data-level="11.5" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-data-limitations"><i class="fa fa-check"></i><b>11.5</b> ATAC-Seq data <strong>limitations</strong>:</a></li>
<li class="chapter" data-level="11.6" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-data-considerations"><i class="fa fa-check"></i><b>11.6</b> ATAC-Seq data considerations</a></li>
<li class="chapter" data-level="11.7" data-path="atac-seq.html"><a href="atac-seq.html#atac-seq-analysis-tools"><i class="fa fa-check"></i><b>11.7</b> ATAC-seq analysis tools</a></li>
<li class="chapter" data-level="11.8" data-path="atac-seq.html"><a href="atac-seq.html#more-resources-about-atac-seq-data"><i class="fa fa-check"></i><b>11.8</b> More resources about ATAC-seq data</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="chip-seq.html"><a href="chip-seq.html"><i class="fa fa-check"></i><b>12</b> ChIP-Seq</a>
<ul>
<li class="chapter" data-level="12.1" data-path="chip-seq.html"><a href="chip-seq.html#learning-objectives-10"><i class="fa fa-check"></i><b>12.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="12.2" data-path="chip-seq.html"><a href="chip-seq.html#what-are-the-goals-of-chip-seq-analysis"><i class="fa fa-check"></i><b>12.2</b> What are the goals of ChIP-Seq analysis?</a></li>
<li class="chapter" data-level="12.3" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-general-workflow-overview"><i class="fa fa-check"></i><b>12.3</b> ChIP-Seq general workflow overview</a></li>
<li class="chapter" data-level="12.4" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-data-strengths"><i class="fa fa-check"></i><b>12.4</b> ChIP-Seq data <strong>strengths</strong>:</a></li>
<li class="chapter" data-level="12.5" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-data-limitations"><i class="fa fa-check"></i><b>12.5</b> ChIP-Seq data <strong>limitations</strong>:</a></li>
<li class="chapter" data-level="12.6" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-data-considerations"><i class="fa fa-check"></i><b>12.6</b> ChIP-Seq data considerations</a></li>
<li class="chapter" data-level="12.7" data-path="chip-seq.html"><a href="chip-seq.html#chip-seq-analysis-tools"><i class="fa fa-check"></i><b>12.7</b> ChiP-seq analysis tools</a></li>
<li class="chapter" data-level="12.8" data-path="chip-seq.html"><a href="chip-seq.html#more-resources-about-chip-seq-data"><i class="fa fa-check"></i><b>12.8</b> More resources about ChiP-seq data</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html"><i class="fa fa-check"></i><b>13</b> DNA Methylation Sequencing Analysis Methods</a>
<ul>
<li class="chapter" data-level="13.1" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#learning-objectives-11"><i class="fa fa-check"></i><b>13.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="13.2" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#what-are-the-goals-of-analyzing-dna-methylation"><i class="fa fa-check"></i><b>13.2</b> What are the goals of analyzing DNA methylation?</a></li>
<li class="chapter" data-level="13.3" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-data-considerations"><i class="fa fa-check"></i><b>13.3</b> Methylation data considerations</a>
<ul>
<li class="chapter" data-level="13.3.1" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#beta-values-binomially-distributed"><i class="fa fa-check"></i><b>13.3.1</b> Beta values binomially distributed</a></li>
<li class="chapter" data-level="13.3.2" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#measuring-5mc-andor-5hmc"><i class="fa fa-check"></i><b>13.3.2</b> Measuring 5mC and/or 5hmC</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-data-workflow"><i class="fa fa-check"></i><b>13.4</b> Methylation data workflow</a></li>
<li class="chapter" data-level="13.5" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-tools"><i class="fa fa-check"></i><b>13.5</b> Methylation tools!</a>
<ul>
<li class="chapter" data-level="13.5.1" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#quality-control-1"><i class="fa fa-check"></i><b>13.5.1</b> Quality control</a></li>
<li class="chapter" data-level="13.5.2" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#genome-alignment"><i class="fa fa-check"></i><b>13.5.2</b> Genome Alignment</a></li>
<li class="chapter" data-level="13.5.3" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#methylation-calls"><i class="fa fa-check"></i><b>13.5.3</b> Methylation calls</a></li>
<li class="chapter" data-level="13.5.4" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#find-regions-of-interest"><i class="fa fa-check"></i><b>13.5.4</b> Find regions of interest!</a></li>
<li class="chapter" data-level="13.5.5" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#annotation-of-regions-of-interest"><i class="fa fa-check"></i><b>13.5.5</b> Annotation of regions of interest</a></li>
</ul></li>
<li class="chapter" data-level="13.6" data-path="dna-methylation-sequencing-analysis-methods.html"><a href="dna-methylation-sequencing-analysis-methods.html#more-resources"><i class="fa fa-check"></i><b>13.6</b> More resources</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html"><i class="fa fa-check"></i><b>14</b> General Data Analysis Tools</a>
<ul>
<li class="chapter" data-level="14.1" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#learning-objectives-12"><i class="fa fa-check"></i><b>14.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="14.2" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#command-line-vs-gui"><i class="fa fa-check"></i><b>14.2</b> Command Line vs GUI</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#bash"><i class="fa fa-check"></i><b>14.2.1</b> Bash</a></li>
<li class="chapter" data-level="14.2.2" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#r"><i class="fa fa-check"></i><b>14.2.2</b> R</a></li>
<li class="chapter" data-level="14.2.3" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#python"><i class="fa fa-check"></i><b>14.2.3</b> Python</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#data-visualization-tools"><i class="fa fa-check"></i><b>14.3</b> Data Visualization Tools</a></li>
<li class="chapter" data-level="14.4" data-path="general-data-analysis-tools.html"><a href="general-data-analysis-tools.html#more-resources-1"><i class="fa fa-check"></i><b>14.4</b> More resources</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="annotating-genomes.html"><a href="annotating-genomes.html"><i class="fa fa-check"></i><b>15</b> Annotating Genomes</a>
<ul>
<li class="chapter" data-level="15.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#learning-objectives-13"><i class="fa fa-check"></i><b>15.1</b> Learning Objectives</a></li>
<li class="chapter" data-level="15.2" data-path="annotating-genomes.html"><a href="annotating-genomes.html#what-are-reference-genomes"><i class="fa fa-check"></i><b>15.2</b> What are reference genomes?</a></li>
<li class="chapter" data-level="15.3" data-path="annotating-genomes.html"><a href="annotating-genomes.html#what-are-genome-versions"><i class="fa fa-check"></i><b>15.3</b> What are genome versions?</a></li>
<li class="chapter" data-level="15.4" data-path="annotating-genomes.html"><a href="annotating-genomes.html#what-are-the-different-files"><i class="fa fa-check"></i><b>15.4</b> What are the different files?</a>
<ul>
<li class="chapter" data-level="15.4.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#how-to-download-annotation-files"><i class="fa fa-check"></i><b>15.4.1</b> How to download annotation files</a></li>
</ul></li>
<li class="chapter" data-level="15.5" data-path="annotating-genomes.html"><a href="annotating-genomes.html#considerations-for-annotating-genomic-data"><i class="fa fa-check"></i><b>15.5</b> Considerations for annotating genomic data</a>
<ul>
<li class="chapter" data-level="15.5.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#make-sure-you-have-the-right-file-to-start"><i class="fa fa-check"></i><b>15.5.1</b> Make sure you have the right file to start!</a></li>
<li class="chapter" data-level="15.5.2" data-path="annotating-genomes.html"><a href="annotating-genomes.html#be-consistent-in-your-annotations"><i class="fa fa-check"></i><b>15.5.2</b> Be consistent in your annotations</a></li>
<li class="chapter" data-level="15.5.3" data-path="annotating-genomes.html"><a href="annotating-genomes.html#be-clear-in-your-write-ups"><i class="fa fa-check"></i><b>15.5.3</b> Be clear in your write ups!</a></li>
</ul></li>
<li class="chapter" data-level="15.6" data-path="annotating-genomes.html"><a href="annotating-genomes.html#resources-you-will-need-for-annotation"><i class="fa fa-check"></i><b>15.6</b> Resources you will need for annotation!</a>
<ul>
<li class="chapter" data-level="15.6.1" data-path="annotating-genomes.html"><a href="annotating-genomes.html#annotation-databases"><i class="fa fa-check"></i><b>15.6.1</b> Annotation databases</a></li>
<li class="chapter" data-level="15.6.2" data-path="annotating-genomes.html"><a href="annotating-genomes.html#gui-based-annotation-tools"><i class="fa fa-check"></i><b>15.6.2</b> GUI based annotation tools</a></li>
<li class="chapter" data-level="15.6.3" data-path="annotating-genomes.html"><a href="annotating-genomes.html#command-line-based-tools"><i class="fa fa-check"></i><b>15.6.3</b> Command line based tools</a></li>
<li class="chapter" data-level="15.6.4" data-path="annotating-genomes.html"><a href="annotating-genomes.html#more-resources-about-genome-annotation"><i class="fa fa-check"></i><b>15.6.4</b> More resources about genome annotation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html"><i class="fa fa-check"></i><b>16</b> Genomic Tool Glossary</a>
<ul>
<li class="chapter" data-level="16.1" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#bioconductor"><i class="fa fa-check"></i><b>16.1</b> Bioconductor</a>
<ul>
<li class="chapter" data-level="16.1.1" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#notable-bioconductor-genomics-tools"><i class="fa fa-check"></i><b>16.1.1</b> Notable Bioconductor genomics tools:</a></li>
</ul></li>
<li class="chapter" data-level="16.2" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#civic"><i class="fa fa-check"></i><b>16.2</b> CIViC</a></li>
<li class="chapter" data-level="16.3" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#ctat"><i class="fa fa-check"></i><b>16.3</b> CTAT</a></li>
<li class="chapter" data-level="16.4" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#deepphe"><i class="fa fa-check"></i><b>16.4</b> DeepPhe</a></li>
<li class="chapter" data-level="16.5" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#genetic-cancer-risk-detector-garde"><i class="fa fa-check"></i><b>16.5</b> Genetic Cancer Risk Detector (GARDE)</a></li>
<li class="chapter" data-level="16.6" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#gene-set-enrichment-analysis-gsea"><i class="fa fa-check"></i><b>16.6</b> Gene Set Enrichment Analysis (GSEA)</a></li>
<li class="chapter" data-level="16.7" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#integrative-genomics-viewer-igv"><i class="fa fa-check"></i><b>16.7</b> Integrative Genomics Viewer (IGV)</a></li>
<li class="chapter" data-level="16.8" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#ndex"><i class="fa fa-check"></i><b>16.8</b> NDEx</a></li>
<li class="chapter" data-level="16.9" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#multiassayexperiment"><i class="fa fa-check"></i><b>16.9</b> MultiAssayExperiment</a></li>
<li class="chapter" data-level="16.10" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#opencravat"><i class="fa fa-check"></i><b>16.10</b> OpenCRAVAT</a></li>
<li class="chapter" data-level="16.11" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#webmev"><i class="fa fa-check"></i><b>16.11</b> WebMeV</a></li>
<li class="chapter" data-level="16.12" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#xena"><i class="fa fa-check"></i><b>16.12</b> Xena</a>
<ul>
<li class="chapter" data-level="16.12.1" data-path="genomic-tool-glossary.html"><a href="genomic-tool-glossary.html#questions-xena-can-help-you-answer-include"><i class="fa fa-check"></i><b>16.12.1</b> Questions Xena can help you answer include:</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by: </a> </p>
<p style="text-align:center;"> <a href="https://hutchdatascience.org/"> The Fred Hutch Data Science Lab </a></p>
<a href="http://jhudatascience.org/"><img src="https://jhudatascience.org/images/dasl.png" style=" width: 80%; filter: grayscale(100%); padding-left: 40px; padding-top: 8px; vertical-align: top "</a>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
<p style="padding-left: 40px;"><div class="trapezoid" style = "padding-left: 40px;"><span>  <a href="https://forms.gle/hc8Xt3Y2Znjb6M4Y7"> Click here to provide feedback</a> <img src="assets/itcr_arrow.png" style=" width: 10%" ></span></div></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Choosing Genomics Tools</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=1.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
  <link href="assets/style_ITN.css" rel="stylesheet">
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/itcr_main_image.png">
</div>
<div id="genomic-tool-glossary" class="section level1" number="16">
<h1><span class="header-section-number">Chapter 16</span> Genomic Tool Glossary</h1>
<p>Here’s all the tools that have been mentioned in this course or are otherwise recommended for your use. The list is in alphabetical order.</p>
<div id="bioconductor" class="section level2" number="16.1">
<h2><span class="header-section-number">16.1</span> Bioconductor</h2>
<p>The mission of the Bioconductor project is to develop, support, and disseminate free open source software that facilitates rigorous and reproducible analysis of data from current and emerging biological assays. We are dedicated to building a diverse, collaborative, and welcoming community of developers and data scientists.</p>
<p>Bioconductor uses the R statistical programming language, and is open source and open development. It has two releases each year, and an active user community. Bioconductor is also available as Docker images.</p>
<div id="notable-bioconductor-genomics-tools" class="section level3" number="16.1.1">
<h3><span class="header-section-number">16.1.1</span> Notable Bioconductor genomics tools:</h3>
<ul>
<li><p><a href="https://bioconductor.org/packages/release/bioc/html/annotatr.html">annotatr</a></p></li>
<li><p><a href="https://bioconductor.org/packages/release/bioc/html/ensembldb.html">ensembldb</a></p></li>
<li><p><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicRanges.html">GenomicRanges</a> - useful for manipulating and identifying sequences.</p></li>
<li><p><a href="https://bioconductor.org/packages/3.16/data/annotation/html/GO.db.html">GO.db</a> - Gene ontology annotation</p></li>
<li><p><a href="https://bioconductor.org/packages/release/data/annotation/html/org.Hs.eg.db.html">org.Hs.eg.db</a></p></li>
<li><p><a href="https://www.bioconductor.org/packages/release/bioc/html/Rsamtools.html">RSamtools</a></p></li>
<li><p><a href="https://bioconductor.org/packages/3.16/data/annotation/">A full list of Bioconductors annotation packages</a> - contains annotation for all kinds of species and versions of genomes and transcriptomes.</p></li>
<li><p><a href="https://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html#:~:text=Complex%20heatmaps%20are%20efficient%20to,and%20supports%20various%20annotation%20graphics.">ComplexHeatmap</a></p></li>
<li><p><a href="https://bioconductor.org/packages/MultiAssayExperiment/">MultiAssayExperiment</a></p></li>
<li><p><a href="https://www.bioconductor.org/packages/release/bioc/html/limma.html">limma</a></p></li>
<li><p><a href="https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html">DESEq2</a></p></li>
<li><p><a href="https://www.bioconductor.org/packages/release/bioc/html/edgeR.html">edgeR</a></p></li>
<li><p><a href="https://bioconductor.org/packages/curatedTCGAData/">curatedTCGAData</a></p></li>
<li><p><a href="https://bioconductor.org/packages/cBioPortalData/">cBioPortalData</a></p></li>
<li><p><a href="https://bioconductor.org/packages/SingleCellMultiModal/">SingleCellMultiModal</a>
## Cancer Models</p></li>
</ul>
<p>Patient Derived Cancer Models Finder (www.cancermodels.org) is a cancer research platform that aggregates clinical, genomic and functional data from patient-derived xenografts, organoids and cell lines. The PDCM Finder standardises, harmonises and integrates the complex and diverse data associated with PDCMs for cancer community.</p>
<p>Data types used are model meta data, related clinical metadata from the sample for which the model was derived, e.g. molecular and treatment-based. Data are preprocessed, consistently semantically annotated, harmonised and FAIR.</p>
<p>PDCM Finder contains &gt;6200 models across 13 cancer types, including rare pediatric models (17%) and models from minority ethnic backgrounds (33%), making it the largest free to consumer and open access resource of this kind.
Get started at www.cancermodels.org to browse and query models by cancer type</p>
</div>
</div>
<div id="civic" class="section level2" number="16.2">
<h2><span class="header-section-number">16.2</span> CIViC</h2>
<p><a href="www.civicdb.org">CIViC</a> is a knowledgebase and curation interface for the clinical interpretation of variants in cancer. Evidence is curated from published literature describing the diagnostic, prognostic, predictive, predisposing, oncogenic, or functional role of variants in specific cancer types. Evidence submitted by community curators is revised and moderated by expert editors. Individual evidence is synthesized into gene summaries, variant summaries and variant-disease assertions of specific clinical relevance. Anyone can make use of CIViC knowledge through the open web interface or API. Information on how to use or contribute to CIViC is available in our help docs (docs.civicdb.org). The main distinguishing feature of CIViC compared to similar resources it is total commitment to open data sharing. All data are available in the Public Domain (CC0). The code is available for any use under an MIT license.</p>
</div>
<div id="ctat" class="section level2" number="16.3">
<h2><span class="header-section-number">16.3</span> CTAT</h2>
<p>The Trinity Cancer Transcriptome Analysis Toolkit (CTAT, <a href="https://github.com/NCIP/Trinity_CTAT/wiki" class="uri">https://github.com/NCIP/Trinity_CTAT/wiki</a> ) provides a diverse collection of tools to gain insights into the biology of cancer through the lens of the transcriptome. Using RNA-seq as input, CTAT modules enable detection of mutations, fusion transcripts, copy number aberrations, cancer-specific splicing aberrations, and oncogenic viruses including insertions into the human genome. CTAT uses both read mapping and de novo assembly methods to analyze RNA-seq, leveraging tumor bulk and single cell transcriptomes. CTAT modules provide interactive visualizations as outputs, are easily installed for local execution or run via cloud computing (eg. Terra), have detailed user guides and tutorials, and are well-supported through user forums.</p>
</div>
<div id="deepphe" class="section level2" number="16.4">
<h2><span class="header-section-number">16.4</span> DeepPhe</h2>
<p><a href="https://deepphe.github.io/">DeepPhe: Natural Language Processing Tools for Cancer Research</a></p>
<p>Under development since 2014, the DeepPhe suite of software tools aims to extract deep phenotype information from the Electronic Medical Records from patients with cancer. DeepPhe combines:</p>
<ol style="list-style-type: decimal">
<li>multiple natural language processing (NLP) techniques based on cTAKES,1<br />
</li>
<li>a structured cancer information model including concepts from the NCIT and the HemOnc ontology</li>
<li>a graph data model supporting persistence of extracted details including links between patient data enabling semantically informed interpretation, aggregation, and disaggregation of key attributes,<br />
</li>
<li>visual analytics tools supporting patient- and cohort-level displays of extracted data5 including identification of patients matching key research criteria and the examination of individual patient records such as exploration of links between summary items and supporting text mentions, and<br />
</li>
<li>multiple strategies for use, including containerized REST services and GUIs for installation and pipeline execution.</li>
</ol>
<p>DeepPhe tools are available for download and installation from the <a href="https://deepphe.github.io/">DeepPhe website</a> under an open-source license for non-commercial use.</p>
</div>
<div id="genetic-cancer-risk-detector-garde" class="section level2" number="16.5">
<h2><span class="header-section-number">16.5</span> Genetic Cancer Risk Detector (GARDE)</h2>
<p><a href="https://reimagineehr.utah.edu/innovations/garde/">Genetic Cancer Risk Detector (GARDE)</a> screens and identifies patients who meet National Comprehensive Cancer Network (NCCN) criteria for genetic evaluation of familial cancer risk based on their family history in the EHR using both structured data and natural language processing of free-text data. Patients identified by GARDE are imported into an EHR’s population health management dashboard (e.g., Epic’s Healthy Planet module) where genetic counseling staff review individual cases, select, and send bulk outreach messages to patients via chatbot and/or through the patient portal.</p>
<p>GARDE is a population clinical decision support (CDS) platform based on Fast Healthcare Interoperability Resources (FHIR) and CDS Hooks standards to support interoperability and logic sharing beyond single vendor solutions.</p>
</div>
<div id="gene-set-enrichment-analysis-gsea" class="section level2" number="16.6">
<h2><span class="header-section-number">16.6</span> Gene Set Enrichment Analysis (GSEA)</h2>
<p><a href="https://www.gsea-msigdb.org/gsea/index.jsp">Gene Set Enrichment Analysis (GSEA)</a> is a method to identify the coordinate activation or repression of groups of genes that share common biological functions, pathways, chromosomal locations, or regulation, thereby distinguishing even subtle differences between phenotypes or cellular states. Gene set-based enrichment analysis is now standard practice for interpreting global transcription profiling experiments and elucidating the biological mechanisms associated with disease and other biological phenotypes of interest. The method is more powerful than typical single-gene approaches to comparing phenotypes, as it can identify sets of genes (e.g., perturbation signatures or molecular pathways) that are coordinately up- or downregulated when each gene in the set may not be significantly differentially expressed. The GSEA software provides useful visualizations and reports for the exploration and interpretation of results. GSEA bundles direct access to the Molecular Signatures Database (MSigDB) – a comprehensive curated repository of annotated gene sets representing signatures derived from publications, pathway databases, and other sources of public data; MSigDB can also be used independently.</p>
<p>The website for the GSEA-MSigDB resource can be found at gsea-msigdb.org. To get started with GSEA you can view the GSEA User Guide, and access the GSEA software through the downloads page or through the GSEA modules available on GenePattern. See the MSigDB section of the website for more information about MSigDB and to interactively explore the gene sets and their annotations. User support for GSEA and MSigDB is available through our help forum.</p>
</div>
<div id="integrative-genomics-viewer-igv" class="section level2" number="16.7">
<h2><span class="header-section-number">16.7</span> Integrative Genomics Viewer (IGV)</h2>
<p>The <a href="https://software.broadinstitute.org/software/igv/">Integrative Genomics Viewer (IGV)</a> is a track-based browser for interactively exploring genomic data mapped to a reference genome. IGV supports all the standard genomic data types (aligned reads, variants, signal peaks, genome annotations, copy number variation, etc.) as well as sample information, such as clinical, phenotypic, or other attributes. IGV provides great flexibility in loading data, whether investigator generated or publicly available, directly from multiple disparate sources without the need for any pre-processing. Supported data sources include local file systems; web servers on the user’s intranet or the Internet; commercial cloud providers (Google, Amazon, Azure, Dropbox); web links to data in public repositories. Authentication to access private data on the web is supported with the industry standard OAuth protocol.</p>
<p>IGV is available in multiple forms, including both end-user applications and versions for use by developers. The IGV website at <a href="https://igv.org" class="uri">https://igv.org</a> provides access to all modalities of IGV. Download and install the IGV Desktop application from the downloads page. To learn about using the application see the tutorial videos on the IGV YouTube channel and the online User Guide. The IGV-Web app is available at <a href="https://igv.org/app" class="uri">https://igv.org/app</a>. To learn about using the app, the Help link in the menu bar provides access to the documentation, and see also the tutorial videos on the YouTube channel. The igv.js JavaScript component is for web developers who wish to embed IGV in their web apps or portals. More information can be found in the Readme file and the Wiki in the igv.js GitHub repository. IGV user support is available through the igv-help online forum and the GitHub repositories.</p>
</div>
<div id="ndex" class="section level2" number="16.8">
<h2><span class="header-section-number">16.8</span> NDEx</h2>
<p>The <a href="https://www.ndexbio.org/#/">Network Data Exchange (NDEx)</a> project provides an open-source framework where scientists and organizations can store, share and publish biological network knowledge. A distinctive feature of NDEx is that it serves as a home for models that are currently available only as figures, tables, or supplementary information, such as networks produced via systematic mining and integration of large-scale molecular data.</p>
<p>NDEx includes features to support data distribution and access according to FAIR principles. Its full integration with Cytoscape, the popular desktop application for network analysis and visualization, provides the cloud back-end component for data I/O; so, if a network file format can be opened in Cytoscape, it can also be stored in (and retrieved from) NDEx.</p>
<p>NDEx can be accessed via its web user interface or programmatically, via REST API and client libraries in Python, R, Java. Web applications can interface with NDEx via JavaScript: MSigDB, CRAVAT, cBioPortal and IQuery, are all examples of web applications integrated with NDEx.</p>
<p>For more information, please review the About NDEx page.
To get started, visit the NDEx public server: there, you can review the NDEx FAQ, access documentation, contact us, and search or browse thousands of biological network models.</p>
</div>
<div id="multiassayexperiment" class="section level2" number="16.9">
<h2><span class="header-section-number">16.9</span> MultiAssayExperiment</h2>
<p><a href="https://bioconductor.org/packages/MultiAssayExperiment/">MultiAssayExperiment</a> is an R/Bioconductor package that harmonizes data management, manipulation, and subsetting of multiple experimental assays performed on an overlapping set of specimens. It supports on-disk and remote data storage, and provides reshaping tools for adaptability to arbitrary downstream analysis.</p>
<p>MultiAssayExperiment is distinct from alternative approaches in its focus on multi’omic data management and manipulation and in its integration with the Bioconductor ecosystem: it is used by more than 50 other Bioconductor packages, it provides a familiar Bioconductor user experience by extending concepts from SummarizedExperiment while supporting an open-ended mix of data classes for individual assays, and it allows subsetting by genomic ranges, row names, phenotypic data, and assays.</p>
<p>You can get started with the MultiAssayExperiment Bioconductor package documentation, or start with prebuilt MultiAssayExperiments objects from <a href="https://bioconductor.org/packages/curatedTCGAData/">curatedTCGAData</a>, <a href="https://bioconductor.org/packages/cBioPortalData/">cBioPortalData</a>, or <a href="https://bioconductor.org/packages/SingleCellMultiModal/">SingleCellMultiModal</a>.</p>
</div>
<div id="opencravat" class="section level2" number="16.10">
<h2><span class="header-section-number">16.10</span> OpenCRAVAT</h2>
<p><a href="https://run.opencravat.org">OpenCRAVAT</a> uses variation data in many popular variant file formats and its outputs are variant annotations and visualizations. To get started go to opencravat.org. Download and run on your local machine, multi-user servers, at <a href="https://run.opencravat.org" class="uri">https://run.opencravat.org</a> or in the cloud. We offer a broader selection of annotation tools than comparable software and results can be explored with an interactive GUI that provides customized filtering options, interactive tables and widgets. Use it for a single sample or a large cohort, or pull single variant reports with a structured url (Example: <a href="https://run.opencravat.org/webapps/variantreport/index.html?chrom=chr11&amp;pos=48123823&amp;ref_base=A&amp;alt_base=C" class="uri">https://run.opencravat.org/webapps/variantreport/index.html?chrom=chr11&amp;pos=48123823&amp;ref_base=A&amp;alt_base=C</a> )</p>
</div>
<div id="webmev" class="section level2" number="16.11">
<h2><span class="header-section-number">16.11</span> WebMeV</h2>
<p><a href="https://webmev.tm4.org">WebMeV</a> is an online tool that facilitates analysis of large-scale RNA-seq and other multi-omic datasets by providing intuitive access to advanced analytical methods and high-performance computing for a wide range of basic, clinical, and translational researchers. Although WebMeV provides support for “bulk” RNA-seq data, single-cell RNA-seq, and other types of -omic data and provides easy access to public data resources such as The Cancer Genome Atlas (TCGA) and the Genotype-Tissue Expression project (GTEx)—as well as user-provided data. WebMeV uniquely provides a user-friendly, intuitive, interactive interface to processed analytical data uses cloud-computing elasticity for computationally intensive analyses that are increasingly required for genomic data analysis. WebMeV’s design places an emphasis on user-driven data analysis by providing users the ability to visualize, interact with, and dissect genomic data at each step in the analysis with a “point-and-click” interactive data environment. Although the primary input is normalized “count matrices,” WebMeV does include tools for data normalization and quality control and uses Dropbox and Google Drive as means of easily uploading data. Analytical methods include statistical tests for comparing cohorts, for identifying gene seats, for doing functional enrichment analysis on gene sets (GSEA), and for inferring gene regulatory network models and comparing these networks between phenotypes to understand the drivers of disease. WebMeV also provides a platform to support reproducible research and makes code for the entire system and its component methods available as open-source software code.</p>
</div>
<div id="xena" class="section level2" number="16.12">
<h2><span class="header-section-number">16.12</span> Xena</h2>
<p><a href="http://xena.ucsc.edu/">UCSC Xena</a> is a web-based visualization tool for multi-omic data and associated clinical and phenotypic annotations.</p>
<p>Xena showcases seminal cancer genomics datasets from TCGA, the Pan-Cancer Atlas, GDC, PCAWG, ICGC, and more; a total of more than 1500 datasets across 50 cancer types. We support virtually any type of functional genomics data (sometimes known as level 3 or 4 data). This includes SNPs, INDELs, copy number variation, gene expression, ATAC-seq, DNA methylation, exon-, transcript-, miRNA-, lncRNA-expression and structural variants. We also support clinical data such as phenotype information, subtype classifications and biomarkers. All of our data is available for download via python or R APIs, or through our URL links.</p>
<div id="questions-xena-can-help-you-answer-include" class="section level3" number="16.12.1">
<h3><span class="header-section-number">16.12.1</span> Questions Xena can help you answer include:</h3>
<ul>
<li>Is overexpression of this gene associated with better survival?</li>
<li>What genes are differentially expressed between these two groups of samples?</li>
<li>What is the relationship between mutation, copy number, expression, etc for this gene?</li>
</ul>
<p>Our tool differentiates itself by its ability to visualize more uncommon data types, such as DNA methylation, its visual integration of multiple types of genomic data side-by-side, and its ability to easily privately visualize your own data.</p>
<p>Get started with our tutorials: <a href="https://ucsc-xena.gitbook.io/project/tutorials" class="uri">https://ucsc-xena.gitbook.io/project/tutorials</a>. If you use us please cite us: <a href="https://www.nature.com/articles/s41587-020-0546-8" class="uri">https://www.nature.com/articles/s41587-020-0546-8</a></p>

</div>
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All illustrations <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY. </a>
      <br>
      All other materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="annotating-genomes.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="about-the-authors.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
